1. **Scaffold-based molecular design with a graph generative model. Chemical Science 2020** [paper](https://arxiv.org/pdf/1905.13639)
		Lim, J.; Hwang, S.Y.; Moon, S.; Kim, S.; Kim, W.Y. 
		
2.  **DeepGraphMol, a multi-objective, computational strategy for generating molecules with desirable properties: a graph convolution and reinforcement learning approach. BioRxiv 2020** [paper](https://www.biorxiv.org/content/biorxiv/early/2020/06/10/2020.05.25.114165.full.pdf)
		Khemchandani, Y.; O’hagan, S.; Samanta, S.; Swainston, N.; Roberts, T.J.; Bollegala, D.; Kell,
3. **Autonomous discovery in the chemical sciences part II: Outlook.Angewandte Chemie International Edition 2019** [paper](https://arxiv.org/pdf/2003.13755.pdf)
		Coley, C.W.; Eyke, N.S.; Jensen, K.F. 
4. **The power of deep learning to ligand-based novel drug discovery. Expert Opinion on Drug Discovery** 2020. [paper](https://www.tandfonline.com/doi/abs/10.1080/17460441.2020.1745183)
        Baskin, I.I. 
5. **Deep Generative Models for 3D Linker Design.Journal of chemical information and modeling 2020** [paper](https://pubs.acs.org/doi/pdf/10.1021/acs.jcim.9b01120)
        Imrie, F.; Bradley, A.R.; van der Schaar, M.; Deane, C.M. 
6. **Discrete object generation with reversible inductive construction. Advances in Neural Information Processing Systems, 2019** [paper](https://papers.nips.cc/paper/9223-discrete-object-generation-with-reversible-inductive-construction.pdf)
     Seff, A.; Zhou, W.; Damani, F.; Doyle, A.; Adams, R.P.

7. **Efficient learning of non-autoregressive graph variational autoencoders for molecular graph generation. Journal of Cheminformatics 2019**[paper](https://link.springer.com/content/pdf/10.1186/s13321-019-0396-x.pdf)
    Kwon, Y.; Yoo, J.; Choi, Y.S.; Son, W.J.; Lee, D.; Kang, S.
8. **Edge-based sequential graph generation with recurrent neural networks.Neurocomputing 2020**[paper](https://arxiv.org/pdf/2002.00102.pdf)
        Bacciu, D.; Micheli, A.; Podda, M. 
9. **Constrained Bayesian optimization for automatic chemical design using variational autoencoders. Chemical science 2020** [paper](https://pubs.rsc.org/en/content/articlepdf/2020/sc/c9sc04026a)
        Griffiths, R.R.; Hernández-Lobato, J.M. 
10. **The synthesizability of molecules proposed by generative models. Journal of Chemical Information and Modeling 2020.**[paper](https://arxiv.org/pdf/2002.07007.pdf)
        Gao, W.; Coley, C.W.
11. **Netgan: Generating graphs via random walks.arXiv preprint arXiv:1803.00816 2018.**[paper](https://arxiv.org/pdf/1803.00816.pdf)
        Bojchevski, A.; Shchur, O.; Zügner, D.; Günnemann, S. 
12. **Misc-GAN: A Multi-scale Generative Model for Graphs. Frontiers in Big
Data 2019, 2, 3.**[paper](https://www.frontiersin.org/articles/10.3389/fdata.2019.00003/full)
        Zhou, D.; Zheng, L.; Xu, J.; He, J.
13. **Inverse Design of Nanoporous Crystalline Reticular Materials with Deep Generative Models.**
[paper](https://s3-eu-west-1.amazonaws.com/itempdf74155353254prod/12186681/Inverse_Design_of_Nanoporous_Crystalline_Reticular_Materials_with_Deep_Generative_Models_v1.pdf)
        Yao, Z.; Sanchez-Lengeling, B.; Bobbitt, N.S.; Bucior, B.J.; Kumar, S.G.H.; Collins, S.P.; Burns, T.; Woo, T.K.; Farha, O.; Snurr, R.Q.; others.
14. **Generative graph convolutional network for growing graphs. ICASSP 2019-2019 IEEE International Conference on Acoustics, Speech and Signal Processing (ICASSP). IEEE, 2019, pp. 3167–3171.**[paper](https://arxiv.org/pdf/1903.02640.pdf)
         Xu, D.; Ruan, C.; Motwani, K.; Korpeoglu, E.; Kumar, S.; Achan, K.
15. **Inverse design of porous materials using artificial neural networks. Science advances 2020, 6, eaax9324. Materials 2020, xx, 5 16 of 28**[paper](https://advances.sciencemag.org/content/advances/6/1/eaax9324.full.pdf)
         Kim, B.; Lee, S.; Kim, J.
16. **Exploring the chemical space without bias: data-free molecule generation with DQN and SELFIES.[paper](https://ml4physicalsciences.github.io/files/NeurIPS_ML4PS_2019_153.pdf)**
         Gaudin, T.; Nigam, A.; Aspuru-Guzik, A. 
17. **Junction tree variational autoencoder for molecular graph generation. arXiv preprint arXiv:1802.04364 2018.**[paper](https://arxiv.org/pdf/1802.04364.pdf)  [code](https://github.com/wengong-jin/icml18-jtnn)
        Jin, W.; Barzilay, R.; Jaakkola, T.
18. **Graph convolutional policy network for goal-directed molecular graph generation. Advances in neural information processing systems, 2018, pp. 6410–6421.**[paper](https://papers.nips.cc/paper/7877-graph-convolutional-policy-network-for-goal-directed-molecular-graph-generation.pdf)
        You, J.; Liu, B.; Ying, Z.; Pande, V.; Leskovec, J.
19. **Graphrnn: Generating realistic graphs with deep auto-regressive models. arXiv preprint arXiv:1802.08773 2018.**[paper](https://arxiv.org/pdf/1802.08773.pdf)
      You, J.; Ying, R.; Ren, X.; Hamilton, W.L.; Leskovec, J.
20. **MolGAN: An implicit generative model for small molecular graphs. arXiv:1805.11973 2018**[paper](https://arxiv.org/pdf/1805.11973.pdf)[code](https://github.com/nicola-decao/MolGAN)
        Nicola De Cao and Thomas Kipf
21.**Generating Tertiary Protein Structures via an Interpretative Variational Autoencoder. arXiv preprint arXiv:2004.07119 2020.**[paper](https://arxiv.org/pdf/2004.07119.pdf)
        Guo, X.; Tadepalli, S.; Zhao, L.; Shehu, A.
21. **Learn to Generate Time Series Conditioned Graphs with Generative Adversarial Nets. arXiv preprint arXiv:2003.01436 2020.**[paper](https://arxiv.org/pdf/2003.01436.pdf)
        Yang, S.; Liu, J.; Wu, K.; Li, M.
22. **GEOM: Energy-annotated molecular conformations for property prediction and molecular generation. arXiv preprint arXiv:2006.05531 2020.**[paper](https://arxiv.org/pdf/2006.05531.pdf)
         Axelrod, S.; Gomez-Bombarelli, R. 
23. ** Learning To Navigate The Synthetically Accessible Chemical Space Using Reinforcement Learning. arXiv preprint arXiv:2004.12485 2020.**[paper](https://arxiv.org/pdf/2004.12485.pdf)
         Gottipati, S.K.; Sattarov, B.; Niu, S.; Pathak, Y.; Wei, H.; Liu, S.; Thomas, K.M.; Blackburn, S.; Coley, C.W.; Tang, J.; others.
24. **A model to search for synthesizable molecules. Advances in Neural Information Processing Systems, 2019, pp. 7937–7949.**[paper](http://papers.nips.cc/paper/9007-a-model-to-search-for-synthesizable-molecules.pdf)
        Bradshaw, J.; Paige, B.; Kusner, M.J.; Segler, M.; Hernández-Lobato, J.M.
25. **Deep learning for molecular design—a review of the state of the art. Molecular Systems Design & Engineering 2019, 4, 828–849.**[paper](https://pubs.rsc.org/en/content/getauthorversionpdf/c9me00039a)
        Elton, D.C.; Boukouvalas, Z.; Fuge, M.D.; Chung, P.W.
26. **GuacaMol: benchmarking models for de novo molecular design. Journal of chemical information and modeling 2019, 59, 1096–1108.**[paper](https://pubs.acs.org/doi/pdf/10.1021/acs.jcim.8b00839)
        Brown, N.; Fiscato, M.; Segler, M.H.; Vaucher, A.C.
27. **GraphGen: A Scalable Approach to Domain-agnostic Labeled Graph Generation. Proceedings of The Web Conference 2020, 2020, pp. 1253–1263.**[paper](https://arxiv.org/pdf/2001.08184.pdf)
        Goyal, N.; Jain, H.V.; Ranu, S.
28. **Self-supervised Learning: Generative or Contrastive. arXiv 2020, pp. arXiv–2006.**[paper](https://arxiv.org/pdf/2006.08218.pdf)
        Liu, X.; Zhang, F.; Hou, Z.; Wang, Z.; Mian, L.; Zhang, J.; Tang, J. 
29. **Interpretable Deep Graph Generation with Node-Edge Co-Disentanglement. arXiv preprint arXiv:2006.05385 2020.**[paper](https://arxiv.org/pdf/2006.05385.pdf)
        Guo, X.; Zhao, L.; Qin, Z.; Wu, L.; Shehu, A.; Ye, Y.
30. **Learning Localized Representations of Point Clouds with Graph-Convolutional Generative Adversarial Networks. IEEE Transactions on Multimedia 2020.**[paper](https://ieeexplore.ieee.org/abstract/document/9016091)
		Diego Valsesia ; Giulia Fracastoro ; Enrico Magli
31. **Graph Generators: State of the Art and Open Challenges. ACM Computing Surveys (CSUR) 2020, 53, 1–30.**[paper](https://arxiv.org/pdf/2001.07906.pdf)
        Bonifati, A.; Holubová, I.; Prat-Pérez, A.; Sakr, S.
32. **Graph2Plan: Learning Floorplan Generation from Layout Graphs. arXiv preprint arXiv:2004.13204 2020.**[paper](https://arxiv.org/pdf/2004.13204.pdf)
        Hu, R.; Huang, Z.; Tang, Y.; van Kaick, O.; Zhang, H.; Huang, H. 
33. **Permutation Invariant Graph Generation via Score-Based Generative Modeling. arXiv preprint arXiv:2003.00638 2020.**[paper](https://arxiv.org/pdf/2003.00638.pdf)
        Niu, C.; Song, Y.; Song, J.; Zhao, S.; Grover, A.; Ermon, S.
34. **35.Deep Graph Generators. arXiv preprint arXiv:2006.04159 2020.**[paper](https://arxiv.org/pdf/2006.04159.pdf)
        Stier, J.; Granitzer, M.
35. **36.Deep generative models for generating labeled graphs 2019.**[paper](https://openreview.net/pdf?id=ryg3XILFdN)
        Fan, S.; Huang, B. 
36. **Conditional structure generation through graph variational generative adversarial nets. Advances in Neural Information Processing Systems, 2019, pp. 1340–1351.**[paper](https://papers.nips.cc/paper/8415-conditional-structure-generation-through-graph-variational-generative-adversarial-nets.pdf)
        Yang, C.; Zhuang, P.; Shi, W.; Luu, A.; Li, P.
37. **Designing random graph models using variational autoencoders with applications to chemical design. arXiv preprint arXiv:1802.05283 2018.**[paper](
        Samanta, B.; De, A.; Ganguly, N.; Gomez-Rodriguez, M.
38. **Efficient graph generation with graph recurrent attention networks. Advances in Neural Information Processing Systems, 2019, pp. 4255–4265.**[paper](https://papers.nips.cc/paper/8678-efficient-graph-generation-with-graph-recurrent-attention-networks.pdf)
        Liao, R.; Li, Y.; Song, Y.; Wang, S.; Hamilton, W.; Duvenaud, D.K.; Urtasun, R.; Zemel, R. 
39. **Graphite: Iterative generative modeling of graphs. International Conference on Machine Learning, 2019, pp. 2434–2444.**[paper](http://proceedings.mlr.press/v97/grover19a/grover19a.pdf)
        Grover, A.; Zweig, A.; Ermon, S.
40. **Graphvae: Towards generation of small graphs using variational autoencoders. International Conference on Artificial Neural Networks. Springer, 2018, pp. 412–422. Materials 2020, xx, 5 17 of 28**[paper](https://arxiv.org/pdf/1802.03480.pdf)
        Simonovsky, M.; Komodakis, N.
41. **Deep learning on graphs: A survey. IEEE Transactions on Knowledge and Data Engineering 2020.**[paper](https://arxiv.org/pdf/1812.04202.pdf%E3%80%82)
        Zhang, Z.; Cui, P.; Zhu, W.
42. **Graph neural networks: A review of methods and applications. arXiv preprint arXiv:1812.08434 2018.**[paper](https://arxiv.org/pdf/1812.08434.pdf?source=post_page---------------------------)
        Zhou, J.; Cui, G.; Zhang, Z.; Yang, C.; Liu, Z.; Wang, L.; Li, C.; Sun, M.
43. **A comprehensive survey on graph neural networks. IEEE Transactions on Neural Networks and Learning Systems 2020.**[paper](https://arxiv.org/pdf/1901.00596.pdf)
        Wu, Z.; Pan, S.; Chen, F.; Long, G.; Zhang, C.; Philip, S.Y.
44. **AutoLinker: Automatic Fragment Linking with Deep Conditional Transformer Neural Networks 2020.**[paper](http://itempdf74155353254prod.s3.amazonaws.com/12271508/AutoLinker__Automatic_Fragment_Linking_with_Deep_Conditional_Transformer_Neural_Networks_v1.pdf)
        Yang, Y.; Zheng, S.; Su, S.; Xu, J.; Chen, H.
45. **Graphgan: Graph representation learning with generative adversarial nets. Thirty-second AAAI conference on artificial intelligence, 2018.**[paper](https://www.aaai.org/ocs/index.php/AAAI/AAAI18/paper/viewFile/16611/15969)
        Wang, H.; Wang, J.; Wang, J.; Zhao, M.; Zhang, W.; Zhang, F.; Xie, X.; Guo, M. 
46. **Multi-objective de novo drug design with conditional graph generative model. Journal of cheminformatics 2018, 10, 33.**
[paper](https://link.springer.com/content/pdf/10.1186/s13321-018-0287-6.pdf)[code](https://github.com/kevinid/molecule_generator)
        Li, Y.; Zhang, L.; Liu, Z. 
47. **Learning deep generative models of graphs. arXiv preprint arXiv:1803.03324 2018.**[paper](https://arxiv.org/pdf/1803.03324.pdf)
        Li, Y.; Vinyals, O.; Dyer, C.; Pascanu, R.; Battaglia, P.
48. **Generative models for graph-based protein design. Advances in Neural Information Processing Systems, 2019.** [paper](https://papers.nips.cc/paper/9711-generative-models-for-graph-based-protein-design.pdf)   [code](https://github.com/jingraham/neurips19-graph-protein-design)
        Ingraham, J.; Garg, V.; Barzilay, R.; Jaakkola, T. 
49. **Accelerating Protein Design Using Autoregressive Generative Models. bioRxiv 2019, p. 757252.**[paper](https://www.biorxiv.org/content/biorxiv/early/2019/09/05/757252.full.pdf)
        Riesselman, A.J.; Shin, J.E.; Kollasch, A.W.; McMahon, C.; Simon, E.; Sander, C.; Manglik, A.; Kruse, A.C.; Marks, D.S.
50. **De Novo Protein Design for Novel Folds using Guided Conditional Wasserstein Generative Adversarial Networks (gcWGAN). bioRxiv 2019, p. 769919.**[paper](https://www.biorxiv.org/content/biorxiv/early/2019/09/14/769919.full.pdf)
        Karimi, M.; Zhu, S.; Cao, Y.; Shen, Y. 
51. **52.Labeled graph generative adversarial networks. arXiv preprint arXiv:1906.03220 2019.**[paper](https://arxiv.org/pdf/1906.03220.pdf)
        Fan, S.; Huang, B. 
52. **A graph-based genetic algorithm and generative model/Monte Carlo tree search for the exploration of chemical space. Chemical science 2019, 10, 3567–3572.**[paper](https://pubs.rsc.org/en/content/articlepdf/2019/sc/c8sc05372c)
        Jensen, J.H.
53. **EvoMol: a flexible and interpretable evolutionary algorithm for unbiased de novo molecular generation 2020.**[paper](https://assets.researchsquare.com/files/rs-36676/v1_stamped.pdf)
        Leguy, J.; Cauchy, T.; Glavatskikh, M.; Duval, B.; Da Mota, B.
54. **Conditional molecular design with deep generative models. Journal of chemical information and modeling 2018, 59, 43–52.**[paper](https://arxiv.org/pdf/1805.00108.pdf)
        Kang, S.; Cho, K.
55.  **R-MAT: A recursive model for graph mining. Proceedings of the 2004 SIAM International Conference on Data Mining. SIAM, 2004, pp. 442–446.**[paper](https://kilthub.cmu.edu/articles/R-MAT_A_Recursive_Model_for_Graph_Mining/6609113/files/12101195.pdf)
        Chakrabarti, D.; Zhan, Y.; Faloutsos, C.
56. **A gentle introduction to deep learning for graphs. Neural Networks 2020.**[paper](https://arxiv.org/pdf/1912.12693.pdf)
        Bacciu, D.; Errica, F.; Micheli, A.; Podda, M.
57. **A review on generative adversarial networks: Algorithms, theory, and applications. arXiv preprint arXiv:2001.06937 2020.**[paper](https://arxiv.org/pdf/2001.06937.pdf)
        Gui, J.; Sun, Z.; Wen, Y.; Tao, D.; Ye, J. 
58. **De novo molecular design by combining deep autoencoder recurrent neural networks with generative topographic mapping. Journal of chemical information and modeling 2019, 59, 1182–1196.**[paper](https://pubs.acs.org/doi/abs/10.1021/acs.jcim.8b00751)
         Sattarov, B.; Baskin, I.I.; Horvath, D.; Marcou, G.; Bjerrum, E.J.; Varnek, A.
59. **Autonomous molecular design: Then and now. ACS applied materials & interfaces 2019, 11, 24825–24836.**[paper](https://pubs.acs.org/doi/abs/10.1021/acsami.9b01226)
        Dimitrov, T.; Kreisbeck, C.; Becker, J.S.; Aspuru-Guzik, A.; Saikin, S.K.
60. **A de novo molecular generation method using latent vector based generative adversarial network.** Journal of Cheminformatics 2019.[paper](https://jcheminf.biomedcentral.com/articles/10.1186/s13321-019-0397-9)
    Prykhodko, O.; Johansson, S.V.; Kotsias, P.C.; Arús-Pous, J.; Bjerrum, E.J.; Engkvist, O.; Chen, H.
61. **Graphnvp: An invertible flow model for generating molecular graphs.** arXiv 2019.[paper](https://arxiv.org/abs/1905.11600)
    Madhawa, K.; Ishiguro, K.; Nakago, K.; Abe, M. 
62. **Advances and challenges in deep generative models for de novo molecule generation.** Wiley Interdisciplinary Reviews: Computational Molecular Science 2019. [paper](https://doi.org/10.1002/wcms.1395)
    Xue, D.; Gong, Y.; Yang, Z.; Chuai, G.; Qu, S.; Shen, A.; Yu, J.; Liu, Q.
63. **Encoding robust representation for graph generation.** IJCNN 2019. [paper](https://arxiv.org/abs/1809.10851) [code](https://github.com/dmzou/SCAT)
    Zou, D.; Lerman, G. 
64. **D-vae: A variational autoencoder for directed acyclic graphs. NeurIPS 2019**. [paper](https://arxiv.org/abs/1904.11088) [code](https://github.com/muhanzhang/D-VAE)
    Zhang, M.; Jiang, S.; Cui, Z.; Garnett, R.; Chen, Y.
65. **A two-step graph convolutional decoder for molecule generation. arXiv 2019.** [paper](https://arxiv.org/abs/1906.03412)
    Bresson, X.; Laurent, T. 
66. **Machine-Learning-Assisted De Novo Design of Organic Molecules and Polymers: Opportunities and Challenges. Polymers 2020**. [paper](https://www.mdpi.com/2073-4360/12/1/163)
    Chen, G.; Shen, Z.; Iyer, A.; Ghumman, U.F.; Tang, S.; Bi, J.; Chen, W.; Li, Y.
67. **Molecular Generative Model Based on an Adversarially Regularized Autoencoder. Journal of Chemical Information and Modeling 2019**. [paper](https://doi.org/10.1021/acs.jcim.9b00694)
     Hong, S.H.; Ryu, S.; Lim, J.; Kim, W.Y. 
68. **Out-domain examples for generative models. arXiv 2019**. [paper](https://arxiv.org/abs/1903.02926)
    Pasquini, D.; Mingione, M.; Bernaschi, M.
69. **Deep universal graph embedding neural network. arXiv 2019**.[paper](https://arxiv.org/pdf/1909.10086v1.pdf) [code](https://github.com/codeanonymous/deep-universal-graph-embedding-neural-netw)
    Verma, S.; Zhang, Z.L.
70. **A deep generative model for fragment-based molecule generation. arXiv 2020**. [paper](https://arxiv.org/abs/2002.12826)
    Podda, M.; Bacciu, D.; Micheli, A. 
71. **DiPol-GAN: Generating Molecular Graphs Adversarially with Relational Differentiable Pooling**. [paper](https://pdfs.semanticscholar.org/dbc2/424e4ca8585cc13a33de1e904f4a3e11e811.pdf)
    Guarino, M.; Shah, A.; Rivas, P.
72. **OPAL-Net: A Generative Model for Part-based Object Layout Generation. arXiv 2020**. [paper](https://arxiv.org/abs/2006.00190) [code](https://github.com/atmacvit/opalnet)
    Baghel, R.; Sarvadevabhatla, R.K. 
73. **SHADOWCAST: Controlling Network Properties to Explain Graph Generation. arXiv 2020**. [paper](https://arxiv.org/abs/2006.03774) 
    Tann, W.J.W.; Chang, E.C.; Hooi, B.
74. **Deep generative model for 2D structure of natural product. CMU 2019**. [paper](https://www.cs.cmu.edu/~epxing/Class/10708-19/assets/project/final-reports/project21.pdf)
    Anastos, M.; Farhat, N.; Cao, L.
75. **Mol-CycleGAN: a generative model for molecular optimization. Journal of Cheminformatics 2020**. [paper](https://doi.org/10.1186/s13321-019-0404-1) [code](https://github.com/ardigen/mol-cycle-gan)
    Maziarka, Ł.; Pocha, A.; Kaczmarczyk, J.; Rataj, K.; Danel, T.; Warchoł, M.
76. **A deep generative model for graph layout. IEEE Transactions on visualization and
Computer Graphics 2019**. [paper](https://arxiv.org/abs/1904.12225)
    Kwon, O.H.; Ma, K.L.
77. **GraphAF: a flow-based autoregressive model for molecular graph generation. arXiv  2020.** [paper](https://arxiv.org/abs/2001.09382) [code](https://github.com/DeepGraphLearning/GraphAF)
    Shi, C.; Xu, M.; Zhu, Z.; Zhang, W.; Zhang, M.; Tang, J.
78. **Graph generation by sequential edge prediction. ESANN, 2019**. [paper](https://pdfs.semanticscholar.org/8328/5068283133ae2f1b66285c13b820df0e0092.pdf)
    Bacciu, D.; Micheli, A.; Podda, M.
79. **Deep convolutional generative adversarial network (dcGAN) models for screening and design of small molecules targeting cannabinoid receptors. Molecular pharmaceutics 2019**. [paper](https://pubs.acs.org/doi/abs/10.1021/acs.molpharmaceut.9b00500)
    Bian, Y.; Wang, J.; Jun, J.J.; Xie, X.Q.
80. **SMILES-based deep generative scaffold decorator for de-novo drug design. Journal of Cheminformatics 2020**. [paper](https://chemrxiv.org/articles/SMILES-Based_Deep_Generative_Scaffold_Decorator_for_De-Novo_Drug_Design/11638383/1)
    Arús-Pous, J.; Patronov, A.; Bjerrum, E.J.; Tyrchan, C.; Reymond, J.L.; Chen, H.; Engkvist, O.
81. **DeepScaffold: a comprehensive tool for Scaffold-based de novo drug discovery using deep learning. Journal of Chemical Information and Modeling 2019**. [paper](https://doi.org/10.1021/acs.jcim.9b00727) 
    Li, Y.; Hu, J.; Wang, Y.; Zhou, J.; Zhang, L.; Liu, Z.
82. **Continuous graph flow 2019**. [paper](https://openreview.net/forum?id=BkgZSCEtvr)
    Deng, Z.; Nawhal, M.; Meng, L.; Mori, G.
83. **DeepNC: Deep Generative Network Completion. arXiv 2019**. [paper](https://arxiv.org/abs/1907.07381)
    Tran, C.; Shin, W.Y.; Spitz, A.; Gertz, M.
84. **Designing natural product-like virtual libraries using deep molecule generative models. macromolecules 2018**. [paper](https://www.researchgate.net/publication/326668536_Designing_natural_product-like_virtual_libraries_using_deep_molecule_generative_models)
    Li, Y.; Zhou, X.; Liu, Z.; Zhang, L.
85. **Explore Deep Graph Generation 2019**. [paper](http://web.stanford.edu/class/cs224w/project/26420251.pdf)
     Gu, X.
86. **STGGAN: Spatial-temporal Graph Generation. Proceedings of the 27th ACM SIGSPATIAL
International Conference on Advances in Geographic Information Systems, 2019**. [paper](https://dl.acm.org/doi/abs/10.1145/3347146.3363462)
    Zhang, L.
87. **Auto-decoding Graphs. arXiv 2020**. [paper](https://arxiv.org/abs/2006.02879)
    Shah, S.A.; Koltun, V.
88. **Graph Generation with Variational Recurrent Neural Network. arXiv 2019**. [paper](https://arxiv.org/abs/1910.01743)
    Su, S.Y.; Hajimirsadeghi, H.; Mori, G.
89. **Deep Generative Models for Vision, Languages and Graphs. PhD thesis, Duke University, Durham, NC, USA, 2019**. [paper](https://dukespace.lib.duke.edu/dspace/handle/10161/19880)
    Wang, W.
90. **Comprehensive Study of Generative Methods on Drug Discovery. PhD thesis, 2019**. [paper](https://rc.library.uta.edu/uta-ir/handle/10106/28884)
    Xiu, S.; others.
91. **TG-GAN: Deep Generative Models for Continuously-time Temporal Graph Generation. arXiv 2020**. [paper](https://arxiv.org/abs/2005.08323)
    Zhang, L.; Zhao, L.; Qin, S.; Pfoser, D.
92. **Applications of deep-learning in exploiting large-scale and heterogeneous compound data in industrial pharmaceutical research. Frontiers in pharmacology 2019**. [paper](https://www.frontiersin.org/articles/10.3389/fphar.2019.01303/full)
    David, L.; Arús-Pous, J.; Karlsson, J.; Engkvist, O.; Bjerrum, E.J.; Kogej, T.; Kriegl, J.M.; Beck, B.; Chen, H.
93. **Deep learning for deep Chemistry: optimizing the prediction of chemical patterns.
Frontiers in Chemistry 2019**. [paper](https://www.frontiersin.org/articles/10.3389/fchem.2019.00809/full)
    Cova, T.F.; Pais, A.A.
94. **Deep learning for graphs. In Recent Trends in Learning From Data; Springer, 2020.**
[paper](https://www.springer.com/gp/book/9783030438821)
    Bacciu, D.; Micheli, A.
95. **REINVENT 2.0–an AI tool for de novo drug design 2020**.[paper](https://chemrxiv.org/articles/REINVENT_2_0_an_AI_Tool_for_De_Novo_Drug_Design/12058026/2)
    Blaschke, T.; Arús-Pous, J.; Chen, H.; Margreitter, C.; Tyrchan, C.; Engkvist, O.; Papadopoulos, K.; Patronov,A.
96. **Probabilistic Generative Deep Learning for Molecular Design. arXiv 2019**. [paper](https://arxiv.org/abs/1902.05148)
    Chang, D.T.
97. **Hierarchical graph pooling with structure learning. arXiv 2019**. [paper](https://arxiv.org/abs/1911.05954) [code](https://github.com/cszhangzhen/HGP-SL)
    Zhang, Z.; Bu, J.; Ester, M.; Zhang, J.; Yao, C.; Yu, Z.; Wang, C.
98. **Can graph neural networks count substructures? arXiv 2020**. [paper](https://arxiv.org/abs/2002.04025)
    Chen, Z.; Chen, L.; Villar, S.; Bruna, J.
99. **Cascade2vec: Learning Dynamic Cascade Representation by Recurrent Graph Neural Networks. IEEE Access 2019**. [paper](https://ieeexplore.ieee.org/document/8846015)
    Huang, Z.; Wang, Z.; Zhang, R.
100. **Stochastic graph neural networks. ICASSP 2020**. [paper](https://arxiv.org/abs/2006.02684)
    Gao, Z.; Isufi, E.; Ribeiro, A.
101. **Tiered Graph Autoencoders with PyTorch Geometric for Molecular Graphs. arXiv 2019**. [paper](https://arxiv.org/abs/1908.08612)
    Chang, D.T.
102. **Deep Learning for Molecular Graphs with Tiered Graph Autoencoders and Graph Classification. arXiv 2019**. [paper](https://arxiv.org/abs/1910.11390)
    Chang, D.T.
103. **Gnnexplainer: Generating explanations for graph neural networks.  NeurIPS 2019**. [paper](https://arxiv.org/abs/1903.03894) [code](https://github.com/RexYing/gnn-model-explainer)
    Ying, Z.; Bourgeois, D.; You, J.; Zitnik, M.; Leskovec, J.
104. **Coherent comment generation for chinese articles with a graph-to-sequence model. arXiv 2019**. [paper](https://arxiv.org/abs/1906.01231) [code](https://github.com/lancopku/Graph-to-seq-comment-generation)
    Li, W.; Xu, J.; He, Y.; Yan, S.; Wu, Y.; others.
105. **SO (3)-invariance of informed-graph-based deep neural network for anisotropic elastoplastic materials. Computer Methods in Applied Mechanics and Engineering 2020**. [paper](https://www.sciencedirect.com/science/article/pii/S0045782520300578)
    Heider, Y.; Wang, K.; Sun, W.
106. **Graph Neural Network for Hamiltonian-Based Material Property Prediction. arXiv 2020**. [paper](https://arxiv.org/abs/2005.13352)
    Bai, H.; Chu, P.; Tsai, J.Y.; Wilson, N.; Qian, X.; Yan, Q.; Ling, H.
107. **Graph normalizing flows. NeurIPS 2019**. [paper](https://arxiv.org/abs/1905.13177)
    Liu, J.; Kumar, A.; Ba, J.; Kiros, J.; Swersky, K.
108. **Shape-based generative modeling for de novo drug design. Journal of chemical information and modeling 2019**. [paper](https://doi.org/10.1021/acs.jcim.8b00706)
    Skalic, M.; Jiménez, J.; Sabbadin, D.; De Fabritiis, G.
109. **Generative models for automatic chemical design. In Machine Learning Meets Quantum Physics; Springer, 2020**. [paper](https://arxiv.org/pdf/1907.01632.pdf)
    Schwalbe-Koda, D.; Gómez-Bombarelli, R.
110. **Deep reinforcement learning for multiparameter optimization in de novo drug design. Journal of Chemical Information and Modeling 2019**. [paper](https://doi.org/10.1021/acs.jcim.9b00325)
    Ståhl, N.; Falkman, G.; Karlsson, A.; Mathiason, G.; Bostrom, J.
111. **Multi-resolution autoregressive graph-to-graph translation for molecules. arXiv 2019**. [paper](https://chemrxiv.org/articles/Multi-Resolution_Autoregressive_Graph-to-Graph_Translation_for_Molecules/8266745/1)
    Jin, W.; Barzilay, R.; Jaakkola, T.
112. **Scaffold-based molecular design using graph generative model. arXiv 2019**. [paper](https://arxiv.org/abs/1905.13639)
    Lim, J.; Hwang, S.Y.; Kim, S.; Moon, S.; Kim, W.Y.
113. **Automatic chemical design using a data-driven continuous representation of molecules. ACS central science 2018**. [paper](https://doi.org/10.1021/acscentsci.7b00572)
    Gómez-Bombarelli, R.; Wei, J.N.; Duvenaud, D.; Hernández-Lobato, J.M.; Sánchez-Lengeling, B.; Sheberla,D.; Aguilera-Iparraguirre, J.; Hirzel, T.D.; Adams, R.P.; Aspuru-Guzik, A.
114. **Improving chemical autoencoder latent space and molecular de novo generation
diversity with heteroencoders. Biomolecules 2018**. [paper](https://arxiv.org/abs/1806.09300)
    Bjerrum, E.J.; Sattarov, B.
115. **Syntax-directed variational autoencoder for structured data. arXiv 2018**. [paper](https://arxiv.org/abs/1802.08786) [code](https://github.com/Hanjun-Dai/sdvae)
    Dai, H.; Tian, Y.; Dai, B.; Skiena, S.; Song, L.
116. **Efficient enumeration of monocyclic chemical graphs with given path frequencies. Journal of cheminformatics 2014**. [paper](https://pubmed.ncbi.nlm.nih.gov/24955135/)
    Suzuki, M.; Nagamochi, H.; Akutsu, T.
117. **Optimization of molecules via deep reinforcement learning. Scientific reports 2019**. [paper](https://www.nature.com/articles/s41598-019-47148-x) [code](https://github.com/google-research/google-research/tree/master/mol_dqn)
    Zhou, Z.; Kearnes, S.; Li, L.; Zare, R.N.; Riley, P.
118. **Generating focused molecule libraries for drug discovery with recurrent neural networks. ACS central science 2018**. [paper](https://doi.org/10.1021/acscentsci.7b00512)
    Segler, M.H.; Kogej, T.; Tyrchan, C.; Waller, M.P.
119. **ChemTS: an efficient python library for de novo molecular generation. Science and technology of advanced materials 2017**. [paper](https://www.tandfonline.com/doi/full/10.1080/14686996.2017.1401424)
    Yang, X.; Zhang, J.; Yoshizoe, K.; Terayama, K.; Tsuda, K.
120. **Exploring Deep Recurrent Models With Reinforcement Learning for Molecule Design. ICLR Conference 2018***
[pdf](https://openreview.net/pdf?id=HkcTe-bR-) 
		Neil, Daniel and Segler, Marwin and Guasch, Laura and Ahmed, Mohamed and Plumbley, Dean and Sellwood, Matthew and Brown, Nathan

121. **Deep reinforcement learning for de novo drug design. Science Advances. 2018***
 [pdf](https://advances.sciencemag.org/content/4/7/eaap7885.full.pdf)  

	   Popova, Mariya and Isayev, Olexandr and Tropsha, Alexander

122. **Tuning artificial intelligence on the de novo design of natural-product-inspired retinoid X receptor modulators. Communications Chemistry. 2018***
 [pdf](https://www.nature.com/articles/s42004-018-0068-1.pdf)   
		Merk, Daniel and Grisoni, Francesca and Friedrich, Lukas and Schneider, Gisbert*

123. **De novo molecule design by translating from reduced graphs to SMILES. Journal of chemical information and modeling. 2018***
 [pdf](https://pubs.acs.org/doi/pdfplus/10.1021/acs.jcim.8b00626?casa_token=2WeU6DX6WVcAAAAA:YqdC52Zy0YUNlsffg0OG1zsjYrG2Zn3t7a_JZZJXXO3QDSjNmeVqKl9F6RPISAnqI_L35XBYqelAq1w) 
	 Pogány, Peter and Arad, Navot and Genway, Sam and Pickett, Stephen D

124. **Molecular generative model based on conditional variational
autoencoder for de novo molecular design. Journal of Cheminformatics. 2018***
 [pdf](https://jcheminf.biomedcentral.com/track/pdf/10.1186/s13321-018-0286-7)  [code](https://github.com/jaechanglim/CVAE)
	  Lim, Jaechang and Ryu, Seongok and Kim, Jin Woo and Kim, Woo Youn

125. **Prototype-based compound discovery using deep generative models. Molecular Pharmaceutics. 2018***
 [pdf](https://pubs.acs.org/doi/pdf/10.1021/acs.molpharmaceut.8b00474?casa_token=IGr7Iiry0wQAAAAA:TRvGvbfi4-Vos0LDaeu2VFR3z0F00pwWNxv5ZCqyxK0VAZ7CWdLc4Ula2kofmOKhMQr_YngueLa_U-k)  
		Harel, Shahar and Radinsky, Kira

126. **Decoding Molecular Graph Embeddings with Reinforcement Learning. ArXiv preprint. 2019***
 [pdf](https://arxiv.org/pdf/1904.08915.pdf)  
		Kearnes, Steven and Li, Li and Riley, Patrick*

127. **Reinforced adversarial neural computer for de novo molecular design. Journal of chemical information and modeling. 2018***
 [pdf](https://pubs.acs.org/doi/pdf/10.1021/acs.jcim.7b00690)  

		Putin, Evgeny and Asadulaev, Arip and Ivanenkov, Yan and Aladinskiy, Vladimir and Sanchez-Lengeling, Benjamin and Aspuru-Guzik, Alán and Zhavoronkov, Alex*

128. **Adversarial threshold neural computer for molecular de novo design. Molecular Pharmaceutics. 2018***
 [pdf](https://pubs.acs.org/doi/pdfplus/10.1021/acs.molpharmaceut.7b01137)  
		Putin, Evgeny and Asadulaev, Arip and Vanhaelen, Quentin and Ivanenkov, Yan and Aladinskaya, Anastasia V and Aliper, Alex and Zhavoronkov, Alex

129. **Entangled conditional adversarial autoencoder for de novo drug discovery. Molecular Pharmaceutics. 2018***
 [pdf](https://pubs.acs.org/doi/pdf/10.1021/acs.molpharmaceut.8b00839)  
    *Polykovskiy, Daniil and Zhebrak, Alexander and Vetrov, Dmitry and Ivanenkov, Yan and Aladinskiy, Vladimir and Mamoshina, Polina and Bozdaganyan, Marine and Aliper, Alexander and Zhavoronkov, Alex and Kadurin, Artur*

130. **Fast and flexible design of novel proteins using graph neural networks. bioRxiv. 2020***
 [pdf](https://www.biorxiv.org/content/10.1101/868935v2.full.pdf) 
    Strokach, Alexey and Becerra, David and Corbi-Verge, Carles and Perez-Riba, Albert and Kim, Philip M

131. **Image retrieval using scene graphs. IEEE Conference on Computer Vision & Pattern Recognition. 2015***
 [pdf](https://openaccess.thecvf.com/content_cvpr_2015/papers/Johnson_Image_Retrieval_Using_2015_CVPR_paper.pdf)  [code](https://github.com/econser/py_irsg_orig)
	    *Johnson, Justin and Krishna, Ranjay and Stark, Michael and Li, Li-Jia and Shamma, David and Bernstein, Michael and Fei-Fei, Li*

132. **Automatic description generation from images: A survey of models, datasets, and evaluation measures. Journal of Artificial Intelligence Research. 2016***
 [pdf](https://www.jair.org/index.php/jair/article/download/10985/26134/)  
    *Bernardi, Raffaella and Cakici, Ruket and Elliott, Desmond and Erdem, Aykut and Erdem, Erkut and Ikizler-Cinbis, Nazli and Keller, Frank and Muscat, Adrian and Plank, Barbara*

133. **Image generation from scene graphs. IEEE conference
on computer vision and pattern recognition. 2018***  [pdf](https://openaccess.thecvf.com/content_cvpr_2018/papers/Johnson_Image_Generation_From_CVPR_2018_paper.pdf)
    *Johnson, Justin and Gupta, Agrim and Fei-Fei, Li*

134. **Learning from collective intelligence: Feature learning using social images and tags. {ACM transactions on multimedia computing, communications, and applications (TOMM). 2016*** [pdf](https://dl.acm.org/doi/pdf/10.1145/2978656?casa_token=fh4_zAoB_KYAAAAA%3AWhuuMWE4QCi9nUy_Ew1FhAoReTuyoz5BgIIEVw7uHosCkDUGiGa7V9mBBCQ-fcohWH3tVPHhc-Rn)
		Zhang, Hanwang and Shang, Xindi and Luan, Huanbo and Wang, Meng and Chua, Tat-Seng

135. **Interaction networks for learning about objects, relations and physics. Advances in neural information processing systems. 2016***
 [pdf](https://papers.nips.cc/paper/6418-interaction-networks-for-learning-about-objects-relations-and-physics.pdf)  [code](https://github.com/higgsfield/interaction_network_pytorch)
		Battaglia, Peter and Pascanu, Razvan and Lai, Matthew and Rezende, Danilo Jimenez and others*

136. **Convolutional networks on graphs for learning molecular fingerprints. Advances in Neural Information Processing Systems. 2015***
 [pdf](https://papers.nips.cc/paper/5954-convolutional-networks-on-graphs-for-learning-molecular-fingerprints.pdf)  [code](https://github.com/HIPS/neural-fingerprint)
		Duvenaud, David K and Maclaurin, Dougal and Iparraguirre, Jorge and Bombarell, Rafael and Hirzel, Timothy and Aspuru-Guzik, Alán and Adams, Ryan P*

137. **Semi-supervised classification with graph convolutional networks. arXiv. 2016***
 [pdf](https://arxiv.org/pdf/1609.02907.pdf)  [code](https://github.com/tkipf/gcn)
    *Kipf, Thomas N and Welling, Max*

138. **Molecular graph convolutions: moving beyond
fingerprints. Journal of computer-aided molecular design. 2016***  [pdf](https://link.springer.com/content/pdf/10.1007/s10822-016-9938-8.pdf)
    *Kearnes, Steven and McCloskey, Kevin and Berndl, Marc and Pande, Vijay and Riley, Patrick*

139. **Spatial temporal graph convolutional networks for skeleton-based action recognition. Thirty-Second AAAI Conference on Artificial Intelligence. 2018***
 [pdf](https://www.aaai.org/ocs/index.php/AAAI/AAAI18/paper/download/17135/16343)  [code](https://github.com/open-mmlab/mmskeleton)
    *Yan, Sijie and Xiong, Yuanjun and Lin, Dahua*

140. **Graph convolutional neural networks for web-scale recommender systems. Knowledge Discovery & Data Mining. 2018*** [pdf](https://dl.acm.org/doi/pdf/10.1145/3219819.3219890?casa_token=TkOuG5zUrVgAAAAA:XQFAL6IeiSJ9qBAkE05EZWXhMn21AkbMV7X5APIBE5u1uO1ZpnBCq2llxUmxwyyl8YxdnxzOtJZa)
    *Ying, Rex and He, Ruining and Chen, Kaifeng and Eksombatchai, Pong and Hamilton, William L and Leskovec, Jure*

141. **Protein interface prediction using graph convolutional networks. Advances in Neural Information Processing Systems. 2017***
 [pdf](https://papers.nips.cc/paper/7231-protein-interface-prediction-using-graph-convolutional-networks.pdf)  [code](https://github.com/fouticus/pipgcn)
	    *Fout, Alex and Byrd, Jonathon and Shariat, Basir and Ben-Hur, Asa*

142. **Adaptive graph convolutional neural networks. Thirty-second AAAI conference on artificial intelligence. 2018***
 [pdf](https://www.aaai.org/ocs/index.php/AAAI/AAAI18/paper/viewFile/16642/16554)  [code](https://github.com/fouticus/pipgcn)
    *Fout, Alex and Byrd, Jonathon and Shariat, Basir and Ben-Hur, Asa*

143. **Two-stream adaptive graph convolutional networks for skeleton-based action recognition. IEEE Conference on Computer Vision and Pattern Recognition. 2019***
 [pdf](https://openaccess.thecvf.com/content_CVPR_2019/papers/Shi_Two-Stream_Adaptive_Graph_Convolutional_Networks_for_Skeleton-Based_Action_Recognition_CVPR_2019_paper.pdf)  [code](https://github.com/lshiwjx/2s-AGCN)
    *Shi, Lei and Zhang, Yifan and Cheng, Jian and Lu, Hanqing*

144. **An attention enhanced graph convolutional lstm network for skeleton-based action recognition. IEEE conference on computer vision and pattern recognition. 2019***
 [pdf](https://openaccess.thecvf.com/content_CVPR_2019/papers/Si_An_Attention_Enhanced_Graph_Convolutional_LSTM_Network_for_Skeleton-Based_Action_CVPR_2019_paper.pdf)
    *Si, Chenyang and Chen, Wentao and Wang, Wei and Wang, Liang and Tan, Tieniu*

145. **Scene graph generation from objects, phrases and region captions. IEEE International Conference on Computer Vision. 2017***
[pdf](https://openaccess.thecvf.com/content_ICCV_2017/papers/Li_Scene_Graph_Generation_ICCV_2017_paper.pdf)
    *Li, Yikang and Ouyang, Wanli and Zhou, Bolei and Wang, Kun and Wang, Xiaogang*

147. **Defactor: Differentiable edge factorization-based probabilistic graph generation. arXiv. 2018***
 [pdf](https://arxiv.org/pdf/1811.09766.pdf)
   *Assouel, Rim and Ahmed, Mohamed and Segler, Marwin H and Saffari, Amir and Bengio, Yoshua*

148. **Evolving random graph generators: A case for increased algorithmic primitive granularity. 2016 IEEE Symposium Series on Computational Intelligence (SSCI). 2016***
 [pdf](https://web.mst.edu/~tauritzd/pubs/2016/Pope2016a.pdf)
    *Pope, Aaron S and Tauritz, Daniel R and Kent, Alexander D*

149. **Graph r-cnn for scene graph generation. European conference on computer vision (ECCV). 2018***
 [pdf](https://openaccess.thecvf.com/content_ECCV_2018/papers/Jianwei_Yang_Graph_R-CNN_for_ECCV_2018_paper.pdf)  [code](https://github.com/jwyang/graph-rcnn.pytorch)
    *Yang, Jianwei and Lu, Jiasen and Lee, Stefan and Batra, Dhruv and Parikh, Devi*

150. **SHADOWCAST: Controlling Network Properties to Explain Graph Generation. arXiv. 2020***
 [pdf](https://arxiv.org/pdf/2006.03774.pdf)
   *Tann, Wesley Joon-Wie and Chang, Ee-Chien and Hooi, Bryan*

151. **Factorizable net: an efficient subgraph-based framework for scene graph generation. European Conference on Computer Vision (ECCV). 2018***
 [pdf](https://openaccess.thecvf.com/content_ECCV_2018/papers/Yikang_LI_Factorizable_Net_An_ECCV_2018_paper.pdf)  [code](https://github.com/yikang-li/FactorizableNet)
    *Li, Yikang and Ouyang, Wanli and Zhou, Bolei and Shi, Jianping and Zhang, Chao and Wang, Xiaogang*

152. **Ring system-based chemical graph generation for de novo molecular design. Journal of computer-aided molecular design. 2016***
 [pdf](https://link.springer.com/content/pdf/10.1007/s10822-016-9916-1.pdf)
     *Miyao, Tomoyuki and Kaneko, Hiromasa and Funatsu, Kimito*

153. **Unbiased scene graph generation from biased training. IEEE/CVF Conference on Computer Vision and Pattern Recognition. 2020***
 [pdf](https://openaccess.thecvf.com/content_CVPR_2020/papers/Tang_Unbiased_Scene_Graph_Generation_From_Biased_Training_CVPR_2020_paper.pdf)
 [code](https://github.com/KaihuaTang/Scene-Graph-Benchmark.pytorch)
    *Tang, Kaihua and Niu, Yulei and Huang, Jianqiang and Shi, Jiaxin and Zhang, Hanwang*

154. **A constrained molecular graph generation with imposed and forbidden fragments. 9th Hellenic Conference on Artificial Intelligence. 2016***
 [pdf](https://dl.acm.org/doi/pdf/10.1145/2903220.2903234?casa_token=QSX6uK08n9EAAAAA:3tjP0m1PijsOquIDh2yiNjRiDj5YwK1TxlzaizvpXFxEi1XaighdDu_LGkOGLAjv6ysMB1t9wsqP)
    *Omrani, Mohamed Amine and Naanaa, Wady*

155. **Fast random cactus graph generation. International Conference on Supercomputing in Mexico. 2018***
 [pdf](https://link.springer.com/chapter/10.1007/978-3-030-10448-1_12)
    *Trejo-Sánchez, Joel Antonio and Vela-Navarro, Andrés and Flores-Lamas, Alejandro and López-Martínez, José Luis and Bermejo-Sabbagh, Carlos and Cuevas-Cuevas, Nora Leticia and Toral-Cruz, Homero*

156. **Using the Marshall-Olkin extended Zipf distribution in graph generation. European Conference on Parallel Processing. 2015***
 [pdf](https://link.springer.com/content/pdf/10.1007%2F978-3-319-27308-2_40.pdf)
     *Duarte-López, Ariel and Prat-Pérez, Arnau and Pérez-Casany, Marta*

157. **Molecular hypergraph grammar with its application to molecular optimization. International Conference on Machine Learning. 2019***
 [pdf](http://proceedings.mlr.press/v97/kajino19a/kajino19a.pdf)  [code](https://github.com/ibm-research-tokyo/graph_grammar)
    *Kajino, Hiroshi*

158. **Chembo: Bayesian optimization of small organic molecules with synthesizable recommendations. International Conference on Artificial Intelligence and Statistics. 2020***
 [pdf](http://proceedings.mlr.press/v108/korovina20a/korovina20a.pdf)  [code](https://github.com/ks-korovina/chembo)
    *Korovina, Ksenia and Xu, Sailun and Kandasamy, Kirthevasan and Neiswanger, Willie and Poczos, Barnabas and Schneider, Jeff and Xing, Eric*

159. **MolecularRNN: Generating realistic molecular graphs with optimized properties. arXiv. 2019***
 [pdf](https://arxiv.org/pdf/1905.13372.pdf)
     *Popova, Mariya and Shvets, Mykhailo and Oliva, Junier and Isayev, Olexandr*

160. **Symmetry-adapted generation of 3d point sets for the targeted discovery of molecules. Advances in Neural Information Processing Systems. 2019***
 [pdf](https://papers.nips.cc/paper/8974-symmetry-adapted-generation-of-3d-point-sets-for-the-targeted-discovery-of-molecules.pdf)
    *Gebauer, Niklas and Gastegger, Michael and Schütt, Kristof*

161. **Hierarchical Graph-to-Graph Translation for Molecules. ICLR 2020 Conference Blind Submission. 2019***
 [pdf](https://openreview.net/pdf?id=rJeeKTNKDB)  [code](https://github.com/wengong-jin/hgraph2graph)
    *Wengong Jin, Regina Barzilay, Tommi Jaakkola*

162. **A generative model for molecular distance geometry. arXiv. 2019***
 [pdf](https://arxiv.org/pdf/1909.11459.pdf)
    *Simm, Gregor NC and Hernández-Lobato, José Miguel*

163. **MolecularRNN: Generating realistic molecular graphs with optimized properties. DeepAI. 2019***
 [pdf](https://arxiv.org/pdf/1905.13372v1.pdf)
    *Mariya Popova, Mykhailo Shvets, Junier Oliva, Olexandr Isayev*

164. **Graph Deconvolutional Generation. arXiv. 2020***
 [pdf](https://arxiv.org/pdf/2002.07087.pdf)
    *Flam-Shepherd, Daniel and Wu, Tony and Aspuru-Guzik, Alan*

165. **Generative adversarial networks (GAN) based efficient sampling of chemical space for inverse design of inorganic materials. arXiv. 2019***
 [pdf](https://arxiv.org/pdf/1911.05020.pdf)
    *Dan, Yabo and Zhao, Yong and Li, Xiang and Li, Shaobo and Hu, Ming and Hu, Jianjun*

166. **Latent molecular optimization for targeted therapeutic design. arXiv. 2018***
 [pdf](https://arxiv.org/pdf/1809.02032.pdf)
		*Aumentado-Armstrong, Tristan*

167. **Generalization and representational limits of graph neural networks. arXiv. 2020***
 [pdf](https://arxiv.org/pdf/2002.06157.pdf)
    *Garg, Vikas K and Jegelka, Stefanie and Jaakkola, Tommi*

168. **Learning multimodal graph-to-graph translation for molecular optimization. arXiv. 2018***
 [pdf](https://arxiv.org/pdf/1812.01070.pdf)  [code](https://github.com/wengong-jin/iclr19-graph2graph)
    *Jin, Wengong and Yang, Kevin and Barzilay, Regina and Jaakkola, Tommi*

169. **Constrained bayesian optimization for automatic chemical design. arXiv. 2017***
 [pdf](https://arxiv.org/pdf/1709.05501.pdf)  [code](https://github.com/Ryan-Rhys/Constrained-Bayesian-Optimisation-for-Automatic-Chemical-Design)
    *Ryan-Rhys Griffiths, José Miguel Hernández-Lobato*

170. **Concepts of artificial intelligence for computer-assisted drug discovery. Chemical reviews. 2019***
 [pdf](https://pubs.acs.org/doi/pdf/10.1021/acs.chemrev.8b00728)
    *Yang, Xin and Wang, Yifei and Byrne, Ryan and Schneider, Gisbert and Yang, Shengyong*

171. **Adversarial autoencoders with constant-curvature latent manifolds. Applied Soft Computing. 2019***
 [pdf](https://pdf.sciencedirectassets.com/272229/1-s2.0-S1568494619X0006X/1-s2.0-S1568494619302819/main.pdf?X-Amz-Security-Token=IQoJb3JpZ2luX2VjEOX%2F%2F%2F%2F%2F%2F%2F%2F%2F%2FwEaCXVzLWVhc3QtMSJHMEUCIQCZuJv1HYRMWvJ0ksJaLl76Zd93z3mo41OE0XDLfFy5BwIgPEAc4hFo5e2N2C1i1nKo50ho34cySffFKAFLsP483ngqtAMIfhADGgwwNTkwMDM1NDY4NjUiDC%2FSK0kJVTf3lQiDRiqRAyyfs5cO3myV%2FmuoXkrixJd8uE4GwUTyFJ%2FlvInwCwsNxPy4E9ZvMeg%2FB1BxDkMTlzlQCLUyukp%2FqTlOgR%2BieOz%2FJuN7DS1pEGoUHJV0KxCQRkmSP2TBNE3KnOUyuYunaZIL%2Fb99bVXLxKwmKvzi3%2FmkqIEhol%2FOlByg9Dv06POO%2BEUdtpcjBJ8WT98ZUa0ndwX9SSQ8OvcB8ysvO2Nfkg1SDTf82TRHPc5HLzIIaM4Pv%2FcBus9yVJgEbpSG5LfPuGWdTJ9hABagtc7YZ4YdZgLFGSDbtmjWbyzEWMTVKPy96l1nSWvh%2B%2FtVEbYnXFpwTVcDQGDGbbzFu7E8xjXWjiCFdEM9jMAoTKe%2BcHFcrcJJWLUSofLVxdpEkneLmiHSDFnHx7zDVHNOsO5r4MT79p5fmn8l%2F2OOz9NPwOqO%2Fom%2BHeOjxMM%2BQMoHnh6g9ZBuzomMDFbO0PnHVwXr8XviXmQGN4GyXeLrK8t6iBqvT8STvGznrpTiYPvNVsB8tHWOvDES0AJ7gKC9eVx90emK0RBkMISq%2FvcFOusB1rXuOkVops1YQSmdJT%2FZs0SVIzhQdJiE%2BumpiY4aTuIB0PV290qRWxfSbHUlzn2akT%2BIbeguv1kWPkdqDygZTl%2BT78m4dj8klOFmg7Yi8olJKDVknEu4f%2FnSAcfKsWBGkZvjDuxeLHVn1z6lF7cWux4h2JAFwWXPkjQpAaodvR2ibPzXgqZt6U9T7sAuz9R5KEnvwRWHWJcsOnBbQm1M40W%2BI5qgkt4G9v%2BrDJMihXwq%2FQUkawjaO%2FEjkeSci%2FMYilXn48ggNVuvW5KCKikMMd07Td4vG%2B7UhqpUc9shF%2B4Vg%2Ff43oVQKsHfGQ%3D%3D&X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Date=20200703T211659Z&X-Amz-SignedHeaders=host&X-Amz-Expires=300&X-Amz-Credential=ASIAQ3PHCVTYUYFHNNB7%2F20200703%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Signature=5ecb25f06f272a278776f00355027e339e8912a93166f451889cb9f7ebe5baef&hash=9bcb6bb04a6070700ba7eb52a8b44bbc5db20ec0d1a22909399ac48b3baa4e6a&host=68042c943591013ac2b2430a89b270f6af2c76d8dfd086a07176afe7c76c2c61&pii=S1568494619302819&tid=spdf-1254648a-dee7-4e7e-82d8-24b5b8126f61&sid=d50c92a96827534c656b8ea78566e33a8035gxrqa&type=client)  [code](https://github.com/danielegrattarola/ccm-aae)
    *Grattarola, Daniele and Livi, Lorenzo and Alippi, Cesare*

172. **Towards Molecule Generation with Heterogeneous States via Reinforcement Learning. Master Thesis University of Sydney. 2020***
 [pdf](https://ses.library.usyd.edu.au/bitstream/handle/2123/22335/Shi_Thesis.pdf?sequence=4&isAllowed=y)
    *Fangzhou Shi*

173. **Direct steering of de novo molecular generation with descriptor conditional recurrent neural networks. Nature Machine Intelligence. 2020***
 [pdf](https://www.nature.com/articles/s42256-020-0174-5)
    *Panagiotis-Christos Kotsias, Josep Arús-Pous, Hongming Chen, Ola Engkvist, Christian Tyrchan & Esben Jannik Bjerrum*

174. **Artificial Intelligence Based Design of Polymers and Metal Composites: A Perspective . Current Topics in Medicinal Chemistry. 2020***
 [pdf](docserver.ingentaconnect.com/deliver/connect/ben/15680266/v20n11/s2.pdf?expires=1593813680&id=0000&titleid=3902&checksum=9C29BDBE091439A97F31454D4CDA165D)
    *Anju Manuja and Mayank Manuja*

175. **Discovery of Novel Chemical Reactions by Deep Generative Recurrent Neural Network. ChemRxiv. 2020***
 [pdf](https://chemrxiv.org/articles/Discovery_of_Novel_Chemical_Reactions_by_Deep_Generative_Recurrent_Neural_Network/11635929/files/21191058.pdf)
    *Bort, William and Baskin, Igor I and Sidorov, Pavel and Marcou, Gilles and Horvath, Dragos and Madzhidov, Timur and Varnek, Alexandre and Gimadiev, Timur and Nugmanov, Ramil and Mukanov, Artem*

176. **Rethinking drug design in the artificial intelligence era. Nature Reviews Drug Discovery. 2019***
 [pdf](https://www.nature.com/articles/s41573-019-0050-3.pdf)
    *Schneider, Petra and Walters, W Patrick and Plowright, Alleyn T and Sieroka, Norman and Listgarten, Jennifer and Goodnow, Robert A and Fisher, Jasmin and Jansen, Johanna M and Duca, José S and Rush, Thomas S and others*

177. **Structure prediction drives materials discovery. Nature Reviews Materials. 2019***
 [pdf](https://www.repository.cam.ac.uk/bitstream/handle/1810/292161/NRM%20Accepted.pdf?sequence=3)
    *Oganov, Artem R and Pickard, Chris J and Zhu, Qiang and Needs, Richard J

178. **CloseGraph: mining closed frequent graph patterns. Ninth ACM SIGKDD International Conference on Knowledge Discovery and Data Mining. 2003***
 [pdf](https://dl.acm.org/doi/pdf/10.1145/956750.956784?casa_token=f4zdJZWLg-0AAAAA:5pvNRKTIrj0FemhH6A5y71Y34vYgmW8ymChbHLuxB0bzd0C_YxQKXgnn_Liu8Q8vhRvpZvffXdbw)
	   *Yan, Xifeng and Han, Jiawei*

179. **Complex network measures of brain connectivity: uses and interpretations. NeuroImage. 2010***
 [pdf](https://pdf.sciencedirectassets.com/272508/1-s2.0-S1053811910X00131/1-s2.0-S105381190901074X/main.pdf?X-Amz-Security-Token=IQoJb3JpZ2luX2VjEOb%2F%2F%2F%2F%2F%2F%2F%2F%2F%2FwEaCXVzLWVhc3QtMSJHMEUCIDFSYuT1GNAMnAPxHVTghj1sjJ9piwL26mxK1We0m5HvAiEAnCO9hl9L8H4V4QYpMwp3RqepHB7n5CHhSMt4EX1qA%2B8qtAMIfhADGgwwNTkwMDM1NDY4NjUiDCiTT5htckfvYSqGKyqRAywQe%2FYm%2B2T6Dqf1ROoNrJCrQAu4nJ%2FgsoMpj5JlMLWUpq5U39Zzq09WvuH52vgswmWrAqXHhTtquHLf2VtT1uo9%2FcSgZEZ8d%2BpvB6VHllRd%2B7urNSJelg13bWKENi1MJ3fdoZljm6MazRDGnRwMqfuhJLufSeiAK1YVpqjmMquZ8AQWVUGQC03iMdnq3s5plE%2Bt%2BYgc59fc0xhe1sdySb6xhVojKUP24nuVuurSpEI55qiXtkyX67kE5tsqtIosV6eLoc4CCWgHUQ4L0BfwsFQtKcoozQlgpWgqKeCxbWtMGORI0f12js1OBLnmJ0JtbH7ZRlq7toO5S1gjF7d4XwEkhcq8pRcS73VFjbCyXwG%2BaVHnfPGmKcntMaOL2ywLrSiufyeshcSn0if1lBa%2B6i0P3ch0Xmy34kxxqGUJsgP9rkkfXdQj%2Bu9c61oy%2Bv7dI5T5yOlr2CQ6ss39UIk8w8o%2B%2BYKGyP33zNCFDXOwg0CwISErpZbgsE2rmXzmgNcMPrjimLqmMcnJfNorv7bNixgHMKvC%2FvcFOusBEq1stUFLZ53zxG81HHK3t5B7nViJ%2FhV%2BDX7EPB%2FlO9MzRUeCAnSwgB3417qB9PQYSJTuenwJ6yswQuFA2i3hVSkTQaS2peW5uhY2GaOzfb3tvcBnOCubzIGeRVMPUW2o8lm0VVGk%2BY3rRXqSpldpQvE2plD%2FQfGvv%2FxoW%2BJ08QYt7ubBxOUE7NXfkpmuor4Rk8u5gZPO11GRPy6wtEGKUoldwfZ%2F%2FuT3rv2KbT1WrvnyvS8M8q4S8vXoWWP0zV7gsY6VbXMe2eRiYZyufdyQWdQTGjX9hTpLcN2hAA5sAEM5XXtNrnDK72%2Ba8A%3D%3D&X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Date=20200703T214754Z&X-Amz-SignedHeaders=host&X-Amz-Expires=300&X-Amz-Credential=ASIAQ3PHCVTY2XRKLBW4%2F20200703%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Signature=727749f385f85b85c12e6aa7f0d02fda72d10d326c14400d81a3e875ed85897c&hash=222c132351ffb3f25faa82c5ff765812b4f8b1a4608d15b4122077c4ace4d603&host=68042c943591013ac2b2430a89b270f6af2c76d8dfd086a07176afe7c76c2c61&pii=S105381190901074X&tid=spdf-e20c6877-3784-4787-83e6-3e8f8979184b&sid=d50c92a96827534c656b8ea78566e33a8035gxrqa&type=client)   
    *Rubinov, Mikail and Sporns, Olaf*

180. **Graph mining: Laws, generators, and algorithms. ACM computing surveys (CSUR). 2006***
 [pdf](https://dl.acm.org/doi/pdf/10.1145/1132952.1132954?casa_token=yCtw0OOIgXIAAAAA:cyoAXOkUPf7iRvXAWxUwjJ_dEhuwRu04YO1CeXMeyJQUns8Ses49cVO2lSjwLBXrgF8NpWl3oF-j)
    *Chakrabarti, Deepayan and Faloutsos, Christos*

181. **A survey of statistical network models. Now Publishers Inc. 2010**
  [pdf](https://arxiv.org/pdf/0912.5410)  
        *Goldenberg Anna, Zheng Alice X, Fienberg Stephen E, Airoldi Edoardo M*
182. **A random graph model for massive graphs. Proceedings of the thirty-second annual ACM symposium on Theory of computing. 2000**
  [pdf](http://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.78.9450&rep=rep1&type=pdf)  
        *Aiello William, Chung Fan, Lu Linyuan*
183. **Automated generation and analysis of attack graphs. Proceedings 2002 IEEE Symposium on Security and Privacy. 2002**
  [pdf](http://wiki.cac.washington.edu/download/attachments/10000785/Automated+Generation+and+Analysis+of+Attack+Graphs.pdf)  
        *Sheyner Oleg, Haines Joshua, Jha Somesh, Lippmann Richard, Wing Jeannette M*
184. **Kronecker graphs: an approach to modeling networks. Journal of Machine Learning Research. 2010**
  [pdf](https://arxiv.org/pdf/0812.4905)  
        *Leskovec, Jure and Chakrabarti, Deepayan and Kleinberg, Jon and Faloutsos, Christos, Ghahramani Zoubin*
185. **Genetic programming IV: Routine human-competitive machine intelligence. Springer Science & Business Media. 2006**
  [pdf](https://www.researchgate.net/profile/William_Mydlowec/publication/243776894_Genetic_Programming_IV_Routine_Human-Competitive_Machine_Intelligence/links/565eec3c08aefe619b274986/Genetic-Programming-IV-Routine-Human-Competitive-Machine-Intelligence.pdf)    
        *John R. Koza, Martin A. Keane, Matthew J. Streeter, William Mydlowec, Jessen Yu, Guido Lanza*
186. **BRITE: An approach to universal topology generation. MASCOTS 2001, Proceedings Ninth International Symposium on Modeling, Analysis and Simulation of Computer and Telecommunication Systems. 2001**
  [pdf](https://www.academia.edu/download/30406667/2001-003-brite-user-manual.pdf)  
        *Medina Alberto, Lakhina Anukool, Matta Ibrahim, Byers John*
187. **Grammar variational autoencoder. arXiv preprint arXiv:1703.01925. 2017**
  [pdf](https://arxiv.org/pdf/1703.01925)   [code](https://github.com/mkusner/grammarVAE)
        *Matt J. Kusner, Brooks Paige, José Miguel Hernández-Lobato*
188. **Constrained graph variational autoencoders for molecule design. Advances in neural information processing systems. 2018**
  [pdf](https://papers.nips.cc/paper/8005-constrained-graph-variational-autoencoders-for-molecule-design.pdf)   [code](https://github.com/microsoft/constrained-graph-variational-autoencoder)
   *Qi Liu, Miltiadis Allamanis, Marc Brockschmidt, Alexander L. Gaunt*
189. **Molecular sets (MOSES): a benchmarking platform for molecular generation models. arXiv preprint arXiv:1811.12823. 2018**
  [pdf](https://arxiv.org/pdf/1811.12823)   [code](https://github.com/ulamaca/moses)
   *Daniil Polykovskiy, Alexander Zhebrak, Benjamin Sanchez-Lengeling, Sergey Golovanov, Oktai Tatanov, Stanislav Belyaev, Rauf Kurbanov, Aleksey Artamonov, Vladimir Aladinskiy, Mark Veselov, Artur Kadurin, Simon Johansson, Hongming Chen, Sergey Nikolenko, Alan Aspuru-Guzik, Alex Zhavoronkov*
190. **Exploration of databases and methods supporting drug repurposing: a comprehensive survey. Briefings in Bioinformatics. 2020**
  [pdf](https://academic.oup.com/bib/advance-article-pdf/doi/10.1093/bib/bbaa003/32451841/bbaa003.pdf)   
        *Ziaurrehman Tanoli, Umair Seemab, Andreas Scherer, Krister Wennerberg, Jing Tang, Markus Vähä-Koskela*
191. **Inorganic Crystal Structure Database (ICSD). Wiley Online Library. 2019**
  [pdf](https://onlinelibrary.wiley.com/iucr/itc/Ha/ch3o7v0001/sec3o7o4.pdf)  
        *Kaduk JA*
192. **Inverse design of crystals using generalized invertible crystallographic representation. arXiv preprint arXiv:2005.07609 2020** [pdf](https://arxiv.org/pdf/2005.07609.pdf)
       *Ren Zekun, Noh Juhwan, Tian Siyu, Oviedo Felipe, Xing Guangzong, Liang Qiaohao, Aberle Armin, Liu Yi, Li Qianxiao, Jayavelu Senthilnath, others*
193. **Deep Learning Enabled Inorganic Material Generator. ChemRxiv. 2020**
  [pdf](https://chemrxiv.org/articles/Deep_Learning_Enabled_Inorganic_Material_Generator/12312260/files/22695941.pdf) [code](https://github.com/devalab/DING)
        *Pathak Yashaswi, Juneja Karandeep Singh, Varma Girish, Ehara Masahiro, Priyakumar U*
194. **The Generated Databases (GDBs) as a Source of 3D-shaped Building Blocks for Use in Medicinal Chemistry and Drug Discovery. CHIMIA International Journal for Chemistry. 2020**
  [pdf](https://www.ingentaconnect.com/contentone/scs/chimia/2020/00000074/00000004/art00006?crawler=true&mimetype=application/pdf)  
        *Meier Kris1, Bühlmann Sven, Arús-Pous Josep, Reymond Jean-Louis*
195. **USPEX—Evolutionary crystal structure prediction. Computer physics communications. 2006**
  [pdf](https://uspex-team.org/static/file/CPC-USPEX-2006.pdf)  
        *Glass Colin W, Oganov Artem R, Hansen Nikolaus*
196. **Crystal structure prediction via particle-swarm optimization. Physical Review B. 2010**
  [pdf](https://arxiv.org/pdf/1008.3601)  
        *Wang Yanchao, Lv Jian, Zhu Li, Ma Yanming*
197. **CALYPSO: A method for crystal structure prediction. Computer Physics Communications. 2012**
  [pdf](https://arxiv.org/pdf/1205.2264)  
        *Wang Yanchao, Lv Jian, Zhu Li, Ma Yanming*
198. **Inverse design in search of materials with target functionalities. Nature Reviews Chemistry. 2018**
  [pdf](https://www.colorado.edu/faculty/zunger-matter-by-design/sites/default/files/attached-files/689.pdf)  
        *Zunger Alex*
199. **Inverse molecular design using machine learning: Generative models for matter engineering. Science. 2018**
  [pdf](https://science.sciencemag.org/content/361/6400/360.full.pdf)  
        *Benjamin Sanchez-Lengeling, ProfileAlán Aspuru-Guzik*
200. **Machine-enabled inverse design of inorganic solid materials: promises and challenges. Chemical Science. 2020**
  [pdf](https://pubs.rsc.org/en/content/articlepdf/2020/sc/d0sc00594k)  
        *Juhwan Noh, Geun Ho Gu , Sungwon Kim, Yousung Jung*
201. **Study of deep generative models for inorganic chemical compositions. arXiv preprint arXiv:1910.11499. 2019**
  [pdf](https://arxiv.org/pdf/1910.11499)  
        *Sawada Yoshihide, Morikawa Koji, Fujii Mikiya*
202. **Crystalgan: learning to discover crystallographic structures with generative adversarial networks. arXiv preprint arXiv:1810.11203. 2018**
  [pdf](https://arxiv.org/pdf/1810.11203) [code](https://github.com/asmanouira/CrystalGAN)
        *Nouira Asma, Sokolovska Nataliya, Crivello Jean-Claude*
203. **Inverse design of solid-state materials via a continuous representation. Matter. 2019**
  [pdf](https://www.sciencedirect.com/science/article/pii/S2590238519301754/pdfft?md5=75a883da562406a7111b3a247c7b755e&pid=1-s2.0-S2590238519301754-main.pdf) [code](https://github.com/nd-02110114/pytorch-imatgen)
        *Noh Juhwan, Kim Jaehoon, Stein Helge S, Sanchez-Lengeling Benjamin, Gregoire John M, Aspuru-Guzik Alan, Jung Yousung*
204. **Generative adversarial networks for crystal structure prediction. arXiv preprint arXiv:2004.01396. 2020**
  [pdf](https://arxiv.org/pdf/2004.01396)  
        *Sungwon Kim, Juhwan Noh, Geun Ho Gu, Alán Aspuru-Guzik, Yousung Jung*
205. **Commentary: The Materials Project: A materials genome approach to accelerating materials innovation. Apl Materials. 2013**
  [pdf](http://aip.scitation.org/doi/pdf/10.1063/1.4812323)  
        *Anubhav Jain, Shyue Ping Ong, Geoffroy Hautier, Wei Chen, William Davidson Richards, Stephen Dacek, Shreyas Cholia, Dan Gunter, David Skinner, Gerbrand Ceder,  Kristin A. Persson*
206. **The Open Quantum Materials Database (OQMD): assessing the accuracy of DFT formation energies. npj Computational Materials. 2015**
  [pdf](https://www.nature.com/articles/npjcompumats201510.pdf)  
        *Scott Kirklin, James E Saal, Bryce Meredig, Alex Thompson, Jeff W Doak, Muratahan Aykol, Stephan Rühl, Chris Wolverton*
207. **Graph model-based approach to the representation, interpretation, and execution of signal processing systems. International Journal of Intelligent Systems. 1988**
  [pdf](https://dacemirror.sci-hub.st/journal-article/6632851caa4ab2f69cc2dce28b5f9898/sztipanovits1988.pdf)  
        *J. Sztipanovits, G. Karsai, C. Biegl*
208. **Bond Graph model of a PEM fuel cell. Journal of Power Sources. 2006**
  [pdf](https://cyber.sci-hub.st/MTAuMTAxNi9qLmpwb3dzb3VyLjIwMDUuMDguMDQw/saisset2006.pdf)  
        *Rémi Saisset, Guillaume Fontes, Christophe Turpin, Stéphan Astier*
209. **Graph-based semi-supervised learning as a generative model. Carnegie Mellon University. 2007**
  [pdf](https://www.aaai.org/Papers/IJCAI/2007/IJCAI07-401.pdf)  
        *Jingrui He, Jaime Carbonell, Yan Liu*
210. **Generative code modeling with graphs. arXiv preprint arXiv:1805.08490. 2018**
  [pdf](https://arxiv.org/pdf/1805.08490)  
        *Marc Brockschmidt, Miltiadis Allamanis, Alexander L. Gaunt, Oleksandr Polozov*
211. **Deep Learning for Source Code Modeling and Generation: Models, Applications and Challenges. ACM Computing Surveys (CSUR). 2020**
  [pdf](https://arxiv.org/pdf/2002.05442)  
        *Hao Chen, Triet Huynh Minh Le, Muhammad Ali Babar*
212. **Interior Layout Generation Based on Scene Graph and Graph Generation Model. Design Computing and Cognition DCC’20. 2020**
  [pdf](https://www.researchgate.net/profile/Xia_Su3/publication/341440900_Interior_Layout_Generation_Based_on_Scene_Graph_and_Graph_Generation_Model/links/5ec0b8c2458515626cacdd24/Interior-Layout-Generation-Based-on-Scene-Graph-and-Graph-Generation-Model.pdf)  
        *Xia Su, Chenglin Wu, Wen Gao, Weixin Huang*
213. **A Neural Framework for Learning DAG to DAG Translation. 32nd Conference on Neural Information Processing Systems (NIPS 2018). 2018**
  [pdf](http://www.ccs.neu.edu/home/clara/resources/neural-framework-learning.pdf)  
        *M. Clara De Paolis Kaluza, Saeed Amizadeh, Rose Yu*
214. **Attention-Based Graph Evolution. Pacific-Asia Conference on Knowledge Discovery and Data Mining. Pacific-Asia Conference on Knowledge Discovery and Data Mining. 2020**
  [pdf](https://link.springer.com/content/pdf/10.1007%2F978-3-030-47426-3_34.pdf)  
        *Shuangfei Fan, authorBert Huang*
215. **A symmetry-based generative model for shape. 2007 IEEE 11th International Conference on Computer Vision. IEEE. 2007**
  [pdf](http://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.308.8156&rep=rep1&type=pdf)  
        *Nhon H. Trinh, Benjamin B. Kimia*
216. **Edge replacement grammars: A formal language approach for generating graphs. Proceedings of the 2019 SIAM International Conference on Data Mining. 2019**
  [pdf](https://arxiv.org/pdf/1902.07159)  
        *Revanth Reddy, Sarath Chandar, Balaraman Ravindran*
217. **Neural turtle graphics for modeling city road layouts. Proceedings of the IEEE International Conference on Computer Vision. 2019**
  [pdf](http://openaccess.thecvf.com/content_ICCV_2019/papers/Chu_Neural_Turtle_Graphics_for_Modeling_City_Road_Layouts_ICCV_2019_paper.pdf) [code](https://github.com/nv-tlabs/NTG)
        *Hang Chu, Daiqing Li, David Acuna, Amlan Kar, Maria Shugrina, Xinkai Wei, Ming-Yu Liu, Antonio Torralba, Sanja Fidler*
218. **Image-Conditioned Graph Generation for Road Network Extraction. arXiv preprint arXiv:1910.14388. 2019**
  [pdf](https://arxiv.org/pdf/1910.14388) [code](https://github.com/davide-belli/generative-graph-transformer)
        *Davide Belli, Thomas Kipf*
219. **Modelling urban networks using Variational Autoencoders. Applied Network Science. 2019**
  [pdf](https://appliednetsci.springeropen.com/track/pdf/10.1007/s41109-019-0234-0)  
        *Kira Kempinska, Roberto Murcio*
220. **Generative molecular design in low data regimes. Nature Machine Intelligence. 2020**
  [pdf]([https://www.nature.com/articles/s42256-020-0160-y](https://www.nature.com/articles/s42256-020-0160-y)    [code](https://github.com/ETHmodlab/virtual_libraries)
        *Michael Moret, Lukas Friedrich, Francesca Grisoni, Daniel Merk, Gisbert Schneider*
221. **Artificial intelligence in chemistry and drug design. Springer. 2020**
  [pdf](https://link.springer.com/content/pdf/10.1007/s10822-020-00317-x.pdf)  
        *Nathan Brown, Peter Ertl,  Richard Lewis, Torsten Luksch, Daniel Reker, Nadine Schneider*
222. **Learning localized generative models for 3d point clouds via graph convolution. International conference on learning representations. 2018**
  [pdf](https://openreview.net/pdf?id=SJeXSo09FQ)  
        *Diego Valsesia, Giulia Fracastoro, Enrico Magli*
223. **Generative graph prototypes from information theory. IEEE transactions on pattern analysis and machine intelligence. 2015**
  [pdf](https://www.researchgate.net/profile/Edwin_Hancock/publication/281487884_Generative_Graph_Prototypes_from_Information_Theory/links/56206a0a08ae70315b552bf3/Generative-Graph-Prototypes-from-Information-Theory.pdf)  
        *Lin Han, Richard C Wilson, Edwin R Hancock*
224. **The graph neural network model. IEEE Transactions on Neural Networks. 2008**
  [pdf](https://repository.hkbu.edu.hk/cgi/viewcontent.cgi?article=1000&context=vprd_ja)  
        *Franco Scarselli, Marco Gori, Ah Chung Tsoi, Markus Hagenbuchner, Gabriele Monfardini*
225. **Towards variational generation of small graphs. ICLR2018. 2018**
  [pdf](https://openreview.net/pdf?id=rJY7hoi8z)  
        *Martin Simonovsky, Nikos Komodakis*
226. **Dag-gnn: Dag structure learning with graph neural networks. arXiv preprint arXiv:1904.10098. 2019**
  [pdf](https://arxiv.org/pdf/1904.10098) [code](https://github.com/fishmoon1234/DAG-GNN)
        *Yue Yu, Jie Chen, Tian Gao, Mo Yu*
227. **Masked gradient-based causal structure learning. arXiv preprint arXiv:1910.08527. 2019**
  [pdf](https://arxiv.org/pdf/1910.08527)  
        *Ignavier Ng, Zhuangyan Fang, Shengyu Zhu, Zhitang Chen, Jun Wang*
228. **CausalVAE: Structured Causal Disentanglement in Variational Autoencoder. arXiv preprint arXiv:2004.08697. 2020**
  [pdf](https://arxiv.org/pdf/2004.08697)  
        *Mengyue Yang, Furui Liu, Zhitang Chen, Xinwei Shen, Jianye Hao, Jun Wang*
229. **Generative Neural Networks to infer Causal Mechanisms: algorithms and applications. PhD thesis, Université Paris-Saclay. 2019**
  [pdf](https://tel.archives-ouvertes.fr/tel-02528204/document)  
        *Diviyan Kalainathan*
230. **Deep learning for molecular generation. Future medicinal chemistry. 2019**
  [pdf](https://www.researchgate.net/profile/Kangjie_Lin/publication/330740221_Deep_learning_for_molecular_generation/links/5cfdf3be4585157d159fecce/Deep-learning-for-molecular-generation.pdf)  
        *Youjun Xu, Kangjie Lin, Shiwei Wang, Lei Wang, Chenjing Cai, Chen Song, Luhua Lai, Jianfeng Pei*
231. **Scene graph generation with external knowledge and image reconstruction. Proceedings of the IEEE Conference on Computer Vision and Pattern Recognition. 2019**
  [pdf](http://openaccess.thecvf.com/content_CVPR_2019/papers/Gu_Scene_Graph_Generation_With_External_Knowledge_and_Image_Reconstruction_CVPR_2019_paper.pdf) [code](https://github.com/arxrean/SGG_Ex_RC)
        *Jiuxiang Gu, Handong Zhao, Zhe Lin, Sheng Li, Jianfei Cai, Mingyang Ling*
232. **Nevae: A deep generative model for molecular graphs. Proceedings of the AAAI Conference on Artificial Intelligence. 2019**
  [pdf](https://www.aaai.org/ojs/index.php/AAAI/article/download/3903/3781)[code](https://github.com/Networks-Learning/nevae)  
        *Bidisha Samanta, Abir DE, Gourhari Jana, Pratim Kumar Chattaraj, Niloy Ganguly, Manuel Gomez Rodriguez*
233. **Automatic retrosynthetic pathway planning using template-free models. arXiv preprint arXiv:1906.02308. 2019**
  [pdf](https://arxiv.org/pdf/1906.02308)  
        *Kangjie Lin, Youjun Xu, Jianfeng Pei, Luhua Lai*
234. **A deep-learning view of chemical space designed to facilitate drug discovery. arXiv preprint arXiv:2002.02948. 2020**
  [pdf](https://arxiv.org/pdf/2002.02948)  
        *Paul Maragakis, Hunter Nisonoff, Brian Cole, David E. Shaw*
235. **Research and development of MolAICal for drug design via deep learning and classical programming. arXiv preprint arXiv:2006.09747. 2020**
  [pdf](https://arxiv.org/pdf/2006.09747)  
        *Qifeng Bai*
236. **Infinite Machine Creativity. 2020 **
  [pdf](https://labs.sogeti.com/wp-content/uploads/2020/06/SogetiLabs-Infinite-Machine-Creativity.pdf)  
        *Menno van Doorn, Sander Duivestein, Deepa Mamtani, Thijs Pepping*
237. **Bidirectional molecule generation with recurrent neural networks. Journal of chemical information and modeling. 2020**
  [pdf](https://pubs.acs.org/doi/pdf/10.1021/acs.jcim.9b00943)[code](https://github.com/ETHmodlab/BIMODAL)  
        *Francesca Grisoni, Michael Moret, Robin Lingwood, and Gisbert Schneider*
238. **Learning Graph Models for Template-Free Retrosynthesis. arXiv preprint arXiv:2006.07038. 2020**
  [pdf](https://arxiv.org/pdf/2006.07038)  
        *Vignesh Ram Somnath, Charlotte Bunne, Connor W. Coley, Andreas Krause, Regina Barzilay*
239. **Generative network complex for the automated generation of druglike molecules. arXiv preprint arXiv:2005.14286. 2020**
  [pdf](https://arxiv.org/pdf/2005.14286)  
        *Kaifu Gao, Duc D Nguyen, Meihua Tu, Guo-Wei Wei*
240. **SYBA: Bayesian estimation of synthetic accessibility of organic compounds. Journal of Cheminformatics. 2020**
  [pdf](https://link.springer.com/content/pdf/10.1186/s13321-020-00439-2.pdf)    [code](https://github.com/atomistic-machine-learning/schnetpack)
        *Milan Voršilák, Michal Kolář, Ivan Čmelo, Daniel Svozil*

241.  **Scene graph generation with external knowledge and image reconstruction. Proceedings of the IEEE Conference on Computer Vision and Pattern Recognition, 2019,.pp. 1969–1978**
    [pdf](https://arxiv.org/pdf/1904.00560.pdf)
        *Gu, J.; Zhao, H.; Lin, Z.; Li, S.; Cai, J.; Ling, M*
242.   **Nevae: A deep generative model for molecular graphs. Proceedings of the AAAI Conference on Artificial Intelligence, 2019, Vol. 33, pp. 1110–1117.**
    [pdf](https://arxiv.org/pdf/1802.05283.pdf)
        *Samanta, B.; Abir, D.; Jana, G.; Chattaraj, P.K.; Ganguly, N.; Rodriguez, M.G.*
243. **Automatic retrosynthetic pathway planning using template-free models. arXiv preprint arXiv:1906.02308 2019.**
    [pdf](https://arxiv.org/pdf/1906.02308.pdf)
    [code](https://github.com/wengong-jin/hgraph2graph)
        *Lin, K.; Xu, Y.; Pei, J.; Lai, L.*
244. ** A deep-learning view of chemical space designed to facilitate drug discovery. arXiv preprint arXiv:2002.02948 2020.**
    [pdf](https://arxiv.org/pdf/2002.02948.pdf)
        *Maragakis, P.; Nisonoff, H.; Cole, B.; Shaw, D.E.*
245. **Research and development of MolAICal for drug design via deep learning and classical programming. arXiv preprint arXiv:2006.09747 2020.**
    [pdf](https://arxiv.org/pdf/2006.09747.pdf)
        *Bai, Q.*
246. **Infinite Machine Creativity.**
    [pdf](https://labs.sogeti.com/wp-content/uploads/2020/06/SogetiLabs-Infinite-Machine-Creativity.pdf)
        *van Doorn, M.; Duivestein, S.; Mamtani, D.; Pepping, T.*
247. **Bidirectional molecule generation with recurrent neural networks. Journal of chemical information and modeling 2020, 60, 1175–1183.**
    [pdf](https://www.researchgate.net/publication/338413002_Bidirectional_Molecule_Generation_with_Recurrent_Neural_Networks)
        *Grisoni, F.; Moret, M.; Lingwood, R.; Schneider, G.*
248. **Learning Graph Models for Template-Free Retrosynthesis. arXiv preprint arXiv:2006.07038 2020.**
    [pdf](https://arxiv.org/pdf/2006.07038.pdf)
        *Somnath, V.R.; Bunne, C.; Coley, C.W.; Krause, A.; Barzilay, R.*
249. **Generative network complex for the automated generation of druglike molecules. arXiv preprint arXiv:2005.14286 2020.**
    [pdf](https://arxiv.org/pdf/2005.14286.pdf)
        *Gao, K.; Nguyen, D.D.; Tu, M.; Wei, G.W.*
250. Voršilák, M.; Koláˇr, M.; Cmelo, I.; Svozil, D. SYBA: Bayesian estimation of synthetic accessibility of organic ˇ
compounds. Journal of Cheminformatics 2020, 12, 1–13.
251. Grisoni, F.; Schneider, G. De novo Molecular Design with Generative Long Short-term Memory. CHIMIA
International Journal for Chemistry 2019, 73, 1006–1011.
252. Pölsterl, S.; Wachinger, C. Likelihood-Free Inference and Generation of Molecular Graphs. arXiv preprint
arXiv:1905.10310 2019.
253. Jin, W.; Barzilay, R.; Jaakkola, T. Hierarchical Generation of Molecular Graphs using Structural Motifs.
arXiv preprint arXiv:2002.03230 2020.
254. Liu, C.C.; Chan, H.; Luk, K.; Borealis, A. Auto-regressive Graph Generation Modeling with Improved
Evaluation Methods.
Materials 2020, xx, 5 26 of 29
255. Chen, X.; Chen, S.; Zheng, H.; Yao, J.; Cui, K.; Zhang, Y.; Tsang, I.W. Node Attribute Generation on Graphs.
arXiv preprint arXiv:1907.09708 2019.
256. Łukasz, M.; Agnieszka, P.; Kaczmarczyk, J.; Krzysztof, R.; Tomasz, D.; Michał, W. Mol-CycleGAN: a
generative model for molecular optimization. Journal of Cheminformatics 2020, 12.
257. Zou, D.; Lerman, G. Graph Generation via Scattering 2018.
258. Zhang, C.; Lyu, X.; Huang, Y.; Tang, Z.; Liu, Z. Molecular Graph Generation with Deep Reinforced Multitask
Network and Adversarial Imitation Learning. 2019 IEEE International Conference on Bioinformatics and
Biomedicine (BIBM). IEEE, 2019, pp. 326–329.
259. Schwaller, P.; Petraglia, R.; Zullo, V.; Nair, V.H.; Haeuselmann, R.A.; Pisoni, R.; Bekas, C.; Iuliano, A.; Laino,
T. Predicting retrosynthetic pathways using transformer-based models and a hyper-graph exploration
strategy. Chemical Science 2020, 11, 3316–3325.
260. Zang, C.; Wang, F. MoFlow: An Invertible Flow Model for Generating Molecular Graphs. arXiv preprint
arXiv:2006.10137 2020.
261. Mollaysa, A.; Paige, B.; Kalousis, A. Conditional generation of molecules from disentangled representations
2019.
262. Li, X.; Lyu, X.; Zhang, H.; Hu, K.; Tang, Z. Regularizing Variational Autoencoders for Molecular Graph
Generation. International Conference on Neural Information Processing. Springer, 2019, pp. 467–476.
263. Quadri, S.F.; Li, X.; Zheng, D.; Aftab, M.U.; Huang, Y. Multi-Layer Graph Generative Model Using
AutoEncoder for Recommendation Systems. Journal on Big Data 2019, 1, 1.
264. Liu, X.; Tang, S.; Wang, J. Generative Adversarial Graph Representation Learning in Hyperbolic Space.
China Conference on Information Retrieval. Springer, 2018, pp. 119–131.
265. Mansimov, E.; Mahmood, O.; Kang, S.; Cho, K. Molecular geometry prediction using a deep generative
graph neural network. Scientific reports 2019, 9, 1–13.
266. Shine, A.; Kempe, D. Generative Graph Models based on Laplacian Spectra? The World Wide Web
Conference, 2019, pp. 1691–1701.
267. Moreno, S.; Pfeiffer, J.J.; Neville, J. Scalable and exact sampling method for probabilistic generative graph
models. Data Mining and Knowledge Discovery 2018, 32, 1561–1596.
268. Lamb, L.; Garcez, A.; Gori, M.; Prates, M.; Avelar, P.; Vardi, M. Graph Neural Networks Meet
Neural-Symbolic Computing: A Survey and Perspective. arXiv preprint arXiv:2003.00330 2020.
269. Ma, T.; Chen, J.; Xiao, C. Constrained generation of semantically valid graphs via regularizing variational
autoencoders. Advances in Neural Information Processing Systems, 2018, pp. 7113–7124.
270. Das, P.; Sercu, T.; Wadhawan, K.; Padhi, I.; Gehrmann, S.; Cipcigan, F.; Chenthamarakshan, V.; Strobelt, H.;
Santos, C.d.; Chen, P.Y.; others. Accelerating Antimicrobial Discovery with Controllable Deep Generative
Models and Molecular Dynamics. arXiv preprint arXiv:2005.11248 2020.
271. Liang, H.; Stanev, V.; Kusne, A.G.; Takeuchi, I. CRYSPNet: Crystal Structure Predictions via Neural
Network. arXiv preprint arXiv:2003.14328 2020.
272. Jensen, K.F.; Coley, C.W.; Eyke, N.S. Autonomous discovery in the chemical sciences part I: Progress.
Angewandte Chemie International Edition 2019.
273. Oh, S.; Jung, Y.; Kim, S.; Lee, I.; Kang, N. Deep generative design: Integration of topology optimization
and generative models. Journal of Mechanical Design 2019, 141.
274. Takeda, S.; Hama, T.; Hsu, H.H.; Piunova, V.A.; Zubarev, D.; Sanders, D.P.; Pitera, J.W.; Kogoh, M.;
Hongo, T.; Cheng, Y.; others. Molecular Inverse-Design Platform for Material Industries. arXiv preprint
arXiv:2004.11521 2020.
275. Blaschke, T.; Olivecrona, M.; Engkvist, O.; Bajorath, J.; Chen, H. Application of generative autoencoder in
de novo molecular design. Molecular informatics 2018, 37, 1700123.
276. Yoshikawa, N.; Terayama, K.; Sumita, M.; Homma, T.; Oono, K.; Tsuda, K. Population-based de novo
molecule generation, using grammatical evolution. Chemistry Letters 2018, 47, 1431–1434.
277. Saxena, D.; Cao, J. Generative Adversarial Networks (GANs): Challenges, Solutions, and Future Directions.
arXiv preprint arXiv:2005.00065 2020.
278. Salehi, P.; Chalechale, A.; Taghizadeh, M. Generative Adversarial Networks (GANs): An Overview of
Theoretical Model, Evaluation Metrics, and Recent Developments. arXiv preprint arXiv:2005.13178 2020.
279. Dhamala, J.; Ghimire, S.; Sapp, J.L.; Horáˇcek, B.M.; Wang, L. Bayesian optimization on large graphs via
a graph convolutional generative model: Application in cardiac model personalization. International
Materials 2020, xx, 5 27 of 29
Conference on Medical Image Computing and Computer-Assisted Intervention. Springer, 2019, pp.
458–467.
280. Karimi, M.; Hasanzadeh, A.; others. Network-principled deep generative models for designing drug
combinations as graph sets. arXiv preprint arXiv:2004.07782 2020.
281. Gutierrez, J.; Schrum, J. Generative Adversarial Network Rooms in Generative Graph Grammar Dungeons
for The Legend of Zelda. arXiv preprint arXiv:2001.05065 2020.
282. Bacciu, D.; Errica, F.; Micheli, A. Contextual graph markov model: A deep and generative approach to
graph processing. arXiv preprint arXiv:1805.10636 2018.
283. Kawai, W.; Mukuta, Y.; Harada, T. Scalable Generative Models for Graphs with Graph Attention
Mechanism. arXiv preprint arXiv:1906.01861 2019.
284. Singh, V.; Lio, P. Towards probabilistic generative models harnessing graph neural networks for
disease-gene prediction. arXiv preprint arXiv:1907.05628 2019.
285. Shu, D.W.; Park, S.W.; Kwon, J. 3d point cloud generative adversarial network based on tree structured
graph convolutions. Proceedings of the IEEE International Conference on Computer Vision, 2019, pp.
3859–3868.
286. Khodayar, M.; Wang, J.; Wang, Z. Deep Generative Graph Distribution Learning for Synthetic Power Grids.
arXiv preprint arXiv:1901.09674 2019.
287. Broad, T.; Leymarie, F.F.; Grierson, M. Network Bending: Manipulating The Inner Representations of Deep
Generative Models. arXiv preprint arXiv:2005.12420 2020.
288. Geng, Y.; Chen, J.; Chen, Z.; Ye, Z.; Yuan, Z.; Jia, Y.; Chen, H. Generative Adversarial Zero-shot Learning
via Knowledge Graphs. arXiv preprint arXiv:2004.03109 2020.
289. Khan, R.A.; Kleinsteuber, M. Epitomic Variational Graph Autoencoder. arXiv preprint arXiv:2004.01468
2020.
290. Hasanzadeh, A.; Hajiramezanali, E.; Narayanan, K.; Duffield, N.; Zhou, M.; Qian, X. Semi-implicit graph
variational auto-encoders. Advances in neural information processing systems, 2019, pp. 10712–10723.
291. Liang, M.; Meng, Y.; Wang, J.; Lubkeman, D.; Lu, N. FeederGAN: Synthetic Feeder Generation via Deep
Graph Adversarial Nets. arXiv preprint arXiv:2004.01407 2020.
292. Besserve, M.; Sun, R.; Janzing, D.; Schölkopf, B. A theory of independent mechanisms for extrapolation in
generative models. arXiv preprint arXiv:2004.00184 2020.
293. Nauata, N.; Chang, K.H.; Cheng, C.Y.; Mori, G.; Furukawa, Y. House-GAN: Relational Generative
Adversarial Networks for Graph-constrained House Layout Generation. arXiv preprint arXiv:2003.06988
2020.
294. Jing, B.; Chi, E.A.; Tang, J. SGVAE: Sequential Graph Variational Autoencoder. arXiv preprint
arXiv:1912.07800 2019.
295. Wang, W.; Xu, H.; Gan, Z.; Li, B.; Wang, G.; Chen, L.; Yang, Q.; Wang, W.; Carin, L. Graph-driven
generative models for heterogeneous multi-task learning. Proceedings of the AAAI Conference on
Artificial Intelligence, 2020, Vol. 34, pp. 979–988.
296. Gamage, A.; Chien, E.; Peng, J.; Milenkovic, O. Multi-MotifGAN (MMGAN): Motif-Targeted Graph
Generation And Prediction. ICASSP 2020-2020 IEEE International Conference on Acoustics, Speech and
Signal Processing (ICASSP). IEEE, 2020, pp. 4182–4186.
297. Stoehr, N.; Brockschmidt, M.; Stuehmer, J.; Yilmaz, E. Disentangling interpretable generative parameters of
random and real-world graphs. arXiv preprint arXiv:1910.05639 2019.
298. Duan, T.; Lee, J. Graph Embedding VAE: A Permutation Invariant Model of Graph Structure. arXiv preprint
arXiv:1910.08057 2019.
299. Liu, W.; Chen, P.Y.; Yu, F.; Suzumura, T.; Hu, G. Learning graph topological features via GAN. IEEE Access
2019, 7, 21834–21843.
300. Sato, R.; Yamada, M.; Kashima, H. Learning to Sample Hard Instances for Graph Algorithms. arXiv
preprint arXiv:1902.09700 2019.
301.** Graph residual flo[paper](https://arxiv.org/pdf/1909.13521)
         Honda, S.; Akita, H.; Ishiguro, K.; Nakanishi, T.; Oono, K.
302. **Deep Generative Model for Sparse Graphs using Text-Based Learning with Augmentation in Generative Examination Networks. arXiv preprint arXiv:1909.11472 2019.Materials 2020, xx, 5 28 of 29**[paper](https://arxiv.org/pdf/1909.11472)
        van Deursen, R.; Godin, G.
303.** Fast approximate geodesics for deep generative models. International Conference on Artificial Neural Networks. Springer, 2019, pp. 554–566.**
         Chen, N.; Ferroni, F.; Klushyn, A.; Paraschos, A.; Bayer, J.; van der Smagt, P.[paper](https://arxiv.org/pdf/1812.08284)
304.**Learning functional causal models with generative neural networks. In Explainable and Interpretable Models in Computer Vision and Machine Learning; Springer, 2018; pp. 39–80.**[paper](https://arxiv.org/pdf/1709.05321)
         Goudet, O.; Kalainathan, D.; Caillou, P.; Guyon, I.; Lopez-Paz, D.; Sebag, M.
305. **Gan lab: Understanding complex deep generative models using interactive visual experimentation. IEEE transactions on visualization and computer graphics 2018, 25, 1–11.**[paper](https://ieeexplore.ieee.org/iel7/2945/8547224/08440049.pdf)
        Kahng, M.; Thorat, N.; Chau, D.H.P.; Viégas, F.B.; Wattenberg, M. 
306.**Towards quantitative methods to assess network generative models. arXiv preprint arXiv:1809.01369 2018.**[paper](https://arxiv.org/pdf/1809.01369)
         Mostofi, V.; Aliakbary, S.
307. **Semi-supervised learning on graphs with generative adversarial nets. Proceedings of the 27th ACM International Conference on Information and Knowledge Management, 2018, pp. 913–922.**[paper](https://arxiv.org/pdf/1809.00130)
        Ding, M.; Tang, J.; Zhang, J.
308. **Deep structured generative models. arXiv preprint arXiv:1807.03877 2018.**[paper](https://arxiv.org/pdf/1807.03877)
        Xu, K.; Liang, H.; Zhu, J.; Su, H.; Zhang, B.
309.**Deep graph translation. arXiv preprint arXiv:1805.09980 2018.**[paper](https://arxiv.org/pdf/1805.09980)
         Guo, X.; Wu, L.; Zhao, L.
310. **Residual gated graph convnets. arXiv preprint arXiv:1711.07553 2017.**[paper](https://arxiv.org/pdf/1711.07553)
        Bresson, X.; Laurent, T.
311. **301.Generating triples with adversarial networks for scene graph construction.Thirty-Second AAAI Conference on Artificial Intelligence, 2018.**[paper](https://www.aaai.org/ocs/index.php/AAAI/AAAI18/paper/viewFile/17059/16287)
        Klawonn, M.; Heim, E.
312. **Unsupervised generative adversarial cross-modal hashing. Thirty-Second AAAI Conference on Artificial Intelligence, 2018.**[paper](https://www.aaai.org/ocs/index.php/AAAI/AAAI18/paper/download/16746/15730)
        Zhang, J.; Peng, Y.; Yuan, M
313.**A universal marginalizer for amortized inference in generative models. arXiv preprint arXiv:1711.00695 2017.**[paper](https://arxiv.org/pdf/1711.00695)
        Douglas, L.; Zarov, I.; Gourgoulias, K.; Lucas, C.; Hart, C.; Baker, A.; Sahani, M.; Perov, Y.; Johri, S.
314. **Causalgan: Learning causal implicit generative models with adversarial training. arXiv preprint arXiv:1709.02023 2017.**[paper](https://arxiv.org/pdf/1709.02023)
        Kocaoglu, M.; Snyder, C.; Dimakis, A.G.; Vishwanath, S.
315.**Can GAN learn topological features of a graph? arXiv preprint arXiv:1707.06197 2017.**[paper](https://arxiv.org/pdf/1707.06197)
        Liu, W.; Chen, P.Y.; Cooper, H.; Oh, M.H.; Yeung, S.; Suzumura, T.
316.**PULSE: Self-Supervised Photo Upsampling via Latent Space Exploration of Generative Models. Proceedings of the IEEE/CVF Conference on Computer Vision and Pattern Recognition, 2020, pp. 2437–2445.**[paper](http://openaccess.thecvf.com/content_CVPR_2020/papers/Menon_PULSE_Self-Supervised_Photo_Upsampling_via_Latent_Space_Exploration_of_Generative_CVPR_2020_paper.pdf)
        Menon, S.; Damian, A.; Hu, S.; Ravi, N.; Rudin, C.
317. **Contrastive Generative Adversarial Networks. arXiv preprint arXiv:2006.12681 2020.**[paper](https://arxiv.org/pdf/2006.12681)
        Kang, M.; Park, J.
318. **Wasserstein-2 Generative Networks. arXiv preprint arXiv:1909.13082 2019.**[paper](https://arxiv.org/pdf/1909.13082)
Korotin, A.; Egiazarian, V.; Asadulaev, A.; Burnaev, E.
319. **VAEM: a Deep Generative Model for Heterogeneous Mixed Type Data. arXiv preprint arXiv:2006.11941 2020.**[paper](https://arxiv.org/pdf/2006.11941)
        Ma, C.; Tschiatschek, S.; Hernández-Lobato, J.M.; Turner, R.; Zhang, C.
320. **AlphaGAN: Fully Differentiable Architecture Search for Generative Adversarial Networks. arXiv preprint arXiv:2006.09134 2020.**[paper](https://arxiv.org/pdf/2006.09134)
        Tian, Y.; Shen, L.; Su, G.; Li, Z.; Liu, W.
321. **Graph dynamical networks for unsupervised learning of atomic scale dynamics in materials. Nature communications 2019, 10, 1–9.**[paper](https://www.nature.com/articles/s41467-019-10663-6.pdf)
        Xie, T.; France-Lanord, A.; Wang, Y.; Shao-Horn, Y.; Grossman, J.C.
312.  Big-Data Science in Porous Materials: Materials Genomics and Machine Learning. Chemical Reviews
322. **Big-Data Science in Porous Materials: Materials Genomics and Machine Learning. Chemical Reviews 2020.**[paper](https://pubs.acs.org/doi/pdf/10.1021/acs.chemrev.0c00004)
        Jablonka, K.M.; Ongari, D.; Moosavi, S.M.; Smit, B..
323. **Identification of molecules by de novo molecule generation from NMR spectra. Proceedings of the Symposium on Chemoinformatics 42th Materials 2020, xx, 5 28 of 28 Symposium on Chemoinformatics, Tokyo. 2019.**[paper](https://www.jstage.jst.go.jp/article/ciqs/2019/0/2019_1P09/_pdf)[code](https://github.com/john-bradshaw/electro)
        Zhang, J.; Terayama, K.; Sumita, M.; Ito, K.; Kikuchi, J.; Tsuda, K.
324.**On failure modes of molecule generators and optimizers 2020.**
[paper](https://chemrxiv.org/articles/On_Failure_Modes_of_Molecule_Generators_and_Optimizers/12213542/files/22461629.pdf)
        Renz, P.; Van Rompaey, D.; Wegner, J.K.; Hochreiter, S.; Klambauer, G.
325. **Graph convolutional networks for computational drug development and discovery. Briefings in bioinformatics 2020**[paper](https://academic.oup.com/bib/article-pdf/21/3/919/33227266/bbz042.pdf)
        Sun, M.; Zhao, S.; Gilvary, C.; Elemento, O.; Zhou, J.; Wang, F.
326.**An overview of neural networks for drug discovery and the inputs used. Expert opinion on drug discovery 2018**[paper](https://www.tandfonline.com/doi/abs/10.1080/17460441.2018.1547278)
        Xu, Y.; Yao, H.; Lin, K.
327. **Reinforced molecule generation with heterogeneous states. 2019 IEEE International Conference on Data Mining (ICDM). IEEE, 2019**[paper](https://ieeexplore.ieee.org/abstract/document/8970930)
        Shi, F.; You, S.; Xu, C.
329. **Weighted graphs and disconnected components: patterns and a
generator. Proceedings of the 14th ACM SIGKDD international conference on Knowledge discovery and data mining, 2008, pp. 524–532.**[paper](http://www.cs.cmu.edu/~mmcgloho/pubs/butterfly-kdd08.pdf)
        Mcglohon, M.; Akoglu, L.; Faloutsos, C.
330. **A Generative Model For Electron Paths 2019.**[paper](https://arxiv.org/pdf/1805.10970)
        Bradshaw, J.; Kusner, M.J.; Paige, B.; Segler, M.H.S.; Lobato, J.M.H.
331. **A comprehensive survey on geometric deep learning. IEEE Access 2020,
8, 35929–35949.**[paper](https://ieeexplore.ieee.org/iel7/6287639/8948470/09003285.pdf)
        Cao, W.; Yan, Z.; He, Z.; He, Z.
332. **TGG: Transferable Graph Generation for Zero-shot and Few-shot Learning. Proceedings of the 27th ACM International Conference on Multimedia, 2019, pp. 1641–1649.**[paper](https://arxiv.org/pdf/1908.11503)
Zhang, C.; Lyu, X.; Tang, Z.
333. **Open graph benchmark: Datasets for machine learning on graphs. arXiv preprint arXiv:2005.00687 2020.**[paper](https://arxiv.org/pdf/2005.00687)
        Hu, W.; Fey, M.; Zitnik, M.; Dong, Y.; Ren, H.; Liu, B.; Catasta, M.; Leskovec, J.
334. **Isometric Graph Neural Networks. arXiv preprint arXiv:2006.09554 2020.**[paper](https://arxiv.org/pdf/2006.09554)
        Walker, M.; Yan, B.; Xiao, Y.; Wang, Y.; Acharya, A.
335. **Learning Disentangled Representations in Deep Generative Models. IRCL 2017**[paper](https://openreview.net/pdf?id=HJtN5K9gx)
        Siddharth, N.; Paige, B.; Desmaison, A.; van de Meent, J.W.; Wood, F.D.; Goodman, N.D.; Kohli, P.; Torr,
336. **Molecular Optimization Using Graph-to-Graph Translation,arXiv preprint arXiv:1812.01070 2020.**[paper](https://openreview.net/pdf?id=HJtN5K9gx)
        Sandström, E.
337. **Synthesizing designs with interpart dependencies using hierarchical generative adversarial networks. Journal of Mechanical Design 2019, 141.**[paper](https://ideal.umd.edu/assets/pdfs/chen_hgan_idetc_2018.pdf)
        Chen, W.; Fuge, M.
338. **Deep learning–based inverse method for layout design. Structural and Multidisciplinary Optimization 2019, 60, 527–536.**[paper](https://arxiv.org/pdf/1806.03182)
        Zhang, Y.; Ye, W.
339. **Circuit synthesis using generative adversarial networks (GANs). AIAA Scitech 2019 Forum, 2019, p. 2350.**[paper](https://www.engr.colostate.edu/~drherber/files/Guo2019a.pdf)
        Guo, T.; Herber, D.; Allison, J.T.
340. **Wellgan: Generative-adversarial-network-guided well generation for analog/mixed-signal circuit layout. 2019 56th ACM/IEEE Design Automation Conference (DAC). IEEE, 2019, pp. 1–6.**[paper](http://yibolin.com/publications/papers/ANALOG_DAC2019_Xu.pdf)
        Xu, B.; Lin, Y.; Tang, X.; Li, S.; Shen, L.; Sun, N.; Pan, D.Z.
341. **Constraint-Based Circuit Design Using Generative Adversarial Networks. PhD thesis, 2020.** [paper](http://fse.studenttheses.ub.rug.nl/21828/)
        Dalmia, S.
342. **Circuit-GNN: Graph neural networks for distributed circuit design. International Conference on Machine Learning, 2019**[paper](http://proceedings.mlr.press/v97/zhang19e/zhang19e.pdf)
        Zhang, G.; He, H.; Katabi, D.
343.  **DiscoveringSymbolic Models from Deep Learning with Inductive Biases. arXiv preprint arXiv:2006.11287 2020.**[paper](https://arxiv.org/pdf/2006.11287)
        Cranmer, M.; Sanchez-Gonzalez, A.; Battaglia, P.; Xu, R.; Cranmer, K.; Spergel, D.; Ho, S.
